<html>
	<head>
		<link type="text/css" rel="stylesheet" href="css/biovis.css">
		
		<title>BioVis 2011</title>
		<script type="text/javascript">

      var _gaq = _gaq || [];
      _gaq.push(['_setAccount', 'UA-22015250-1']);
      _gaq.push(['_trackPageview']);

      (function() {
        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
      })();

    </script>
	</head>
	
	<body>
		
		<div id="frame">
			
			<div class="header">
			
				<div class="logo">
					<a href="http://www.biovis.net/2011/index.html"><img src="images/logo.png"/></a>
				</div>

				<div class="subtitle">
					<h2>1<sup>st</sup> IEEE Symposium on<br>Biological Data Visualization</h2>					
					<h3>23-24 October 2011 / Providence, RI @ IEEE VisWeek</h3>
				</div>
				
				<div class="navbar">
					<div class="menu">
					<ul>
						<li>
							<a href="http://www.biovis.net/2011/index.html">About</a>
						</li>
						<li>
							<a href="cfp.html" title="Call for Participation">CFP</a>
						</li>
						<li>
							<a href="contest.html">Contest</a>
						</li>						
						<li>
							<a href="submission.html">Submission</a>
						</li>
						<li>
							<a href="program.html">Program</a>
						</li>
						<li>
							<a href="papers_abstracts.html">Papers &amp; Abstracts</a>
						</li>						
						<li>
							<a href="registration.html">Registration</a>
						</li>
						<li>
							<a href="committee.html">Committee</a>
						</li>
					</ul>
					</div>
				</div>
				
			</div>
			
			<div class="content">
			
				<h1>
					Contest
				</h1>

<p>For the BioVis 2011 Symposium, we will inaugurate a new data visualization/analysis contest.  This contest will be focused on specific biological problem domains, and will be based on realistic domain data and domain questions.  For BioVis 2011, the contest involves the analysis of expression Quantitative Trait Locus (eQTL) data.</p>

<h2>Step-by-step Overview</h2>

<ol>
<li>Read the contest description and rules provided below.</li>
<li>Create an account for the <a href="forum/index.html">contest forum</a>. The information provided by you will only be used for the BioVis Data Visualization contest.</li>
<li>Download the <a href="http://biovis.net/forum/index.php">demonstration data set</a> from the corresponding section of the forum. This data is provided in several increasingly realistic versions, from fundamentally clean, to biologically complex.</li>
<li>Develop your approach to a solution using the demonstration data.</li>
<li>Download the <a href="http://biovis.net/forum/index.php">contest data set</a> from the corresponding section of the forum.  This data is provided only in realistically complex versions.</li>
<li>Apply your approach (and tune if necessary) to the contest data and generate results.</li>
<li>Use the <a href="http://biovis.net/forum/index.php">forum</a> to ask the contest organizers for clarifications or to discuss the contest with other participants.</li>
<li>Your contest entry must be received no later than <strong>Sep 7th</strong> (extended from Sep 1st) via email to <a href="mailto:contest@biovis.net">contest@biovis.net</a>.
</ol>


<h2>Contest Specifics</h2>
<ul>
<li><a href="contest.html#general">General Information</a></li>
<li><a href="contest.html#domain">The eQTL biological domain</a></li>
<li><a href="contest.html#data">The eQTL data</a></li>
<li><a href="contest.html#contest">The Challenge</a></li>
<li><a href="contest.html#submission">Submissions</a></li>
<li><a href="contest.html#judging">Judging</a></li>
<li><a href="contest.html#judges">Judges</a></li>
<li><a href="contest.html#awards">Awards</a></li>
<li><a href="contest.html#fineprint">The Fine Print</a></li>
</ul>

<h2>Timeline</h2>
<table>
<tr><td><strong>May 11th</strong></td><td>Contest Announcement and Demonstration Data release</td></tr>
<tr><td><strong>Jun 1st</strong> (approx.)</td><td>Web seminar on contest domain and data</td></tr>
<tr><td><strong>Jul 1st</strong></td><td>Final Contest Data release</td></tr>
<tr><td><strong>Aug 1st</strong></td><td>Bonus Question release</td></tr>
<tr class="highlight"><td><strong>Sep 7th</strong></td><td>Final Submissions due (deadline extended from Sep 1st)</td></tr>
</table>
<br>
<b><i>The information on these pages is subject to amendment and update at any time.  We will post notifications of any changes to the Announcements discussion forum.  Please watch for any changes that may affect your submission</i></b>			

<h2>Contest Contacts</h2>

<table>
<tr><td><strong>Contest General Address</strong></td><td><a href="mailto:contest@biovis.net">contest@biovis.net</a></td></tr>
<tr><td><strong>Contest Chair</strong></td><td><a href="http://www.mathmed.org#William_Ray">William Ray</a></td></tr>
<tr><td><strong>Contest Data Provider<br>&amp; Domain Expert</strong></td><td><a href="http://www.mathmed.org#Christopher_Bartlett">Christopher Bartlett</a></td></tr>
<tr><td><strong>Contest Forum</strong></td><td><a href="http://biovis.net/forum/index.php">Forum</a></td></tr>
</table>


<a name="general"><h2>General Information</h2></a>
<p>
We hope to attract diverse interesting biological problems, and feature a new biological theme each year for the contest.  Through the mechanisms of this contest, we will support three main goals:
</p>

<ol>
<li>The development of a better-informed Vis community, provided with deeper domain-specific intuition into the actual issues of interest to the user community.</li>
<li>A better-tooled biological community, provided with enhanced applications specifically adjusted to meet their analysis needs.</li>
<li>Finally, a mechanism to strongly promote fundable peer collaborations between Visualization and Bio/Life-Sciences researchers.</li>
</ol>

<p>
To meet these goals, each contest will be conducted using a pre-symposium &quot;basic&quot; phase, and, if participant interest is sufficient, a post-symposium &quot;advanced&quot; phase.  Prior to the symposium, data and an explanation of the problem will be made available over the Web.  A web-forum will also be available, where contest participants can discuss the problem and receive clarifications.  At the symposium there will be a session during which contest participants present their results.  After the contest-results presentation there will be a biological tutorial, covering the specific problem being addressed by the contest.  This will be presented by Bio/Life Sciences experts in the domain, including the group directly responsible for developing the contest data set.  The tutorial will explain in detail the complexities of the data, and of the biological intuitions that the end-users are interested in acquiring.  After the symposium there will be a second round of the contest in which a second data set will be approached, using the additional domain-goal intuition acquired from the at-symposium tutorial.  The results of this second round will be presented at a special session of the following year's BioVis symposium.  Outstanding performers in the second round will be invited to co-publish their results with the Bio/Life-science researchers who have helped develop the data set and provided the expert instruction, and will be invited to co-propose fundable projects to the National Science Foundation, National Institutes of Health, or other national or international funding opportunities identified by the data set/problem-domain providers.
</p>

<a name="domain"><h2>The eQTL Biological Domain</h2></a>

<p>The biological domain for the 2011 contest is expression Quantitative Trait Locus data.  eQTL experiments catalog massive collections of correlated genotype and phenotype data, in the hope of detecting important genome-sequence variations that affect phenotypic (specifically RNA expression level) outcomes through non-obvious mechanisms, and of identifying these mechanisms.  Such non-obvious mechanisms typically involve networks of interacting polymorphisms that non-linearly affect specific gene expression levels, conditional on the presence of other polymorphisms, and combinations of polymorphisms, and conditional on the tissue type in which they are acting.</p>

<p>eQTL analysis is a relatively new biological field, because broad-coverage genotype and expression surveys are only just becoming technologically feasible.  It is an attractive field for Visualization and Visual Analytics in the Biological Sciences both because it is a &quot;hot topic&quot; funding priority for the United States National Institutes of Health, with several high-profile Requests for Applications being current for developing analysis capabilities in this area;  And because it is a phenomenally data-rich and information-dense domain (a typical complete analysis is envisioned to survey a few million genotypic loci and tens of thousands of expression levels, differentially in up to 100 tissue types, across 1000 subjects -- Our contest data is a rather restricted subset of this data), and absolutely demands sophisticated mechanisms for summarizing and presenting the data to the end users.  eQTL analysis is also an ideal model system for the emerging field of personalized medicine, because the decision support problem for Personalized Medicine &quot;have I considered all of the important factors in making this decision?&quot; is exactly analogous to the eQTL interaction-network-discovery problem of &quot;have I identified all of the relevant interacting factors?&quot;.
</p>

<p>
Our contest data is generated from actual eQTL analysis data, using an observation-shuffling technique.  This technique preserves the biological complexity of the data, while allowing us to &quot;spike in&quot; specific interaction networks for the purpose of establishing ground truth that should be identifiable.  Because our eventual goals are to encourage and enable the Visualization community to produce tools that are highly relevant to the Bio/Life-Sciences community, it is important that we maintain realistic complexity, and realistic confounding factors within the data.  By maintaining realism, we assure that tools that address the contest data, retain relevance for real data, and we enhance our participant's appreciation of the real complexity and sophistication of the visualization approaches that must be applied. On the flip side, the data contest will encourage the Bio/Life-Sciences community to engage more with the Visualization community given the relevance and the potential utility to the tools and methodology that will result.</p>

<a name="data"><h2>The eQTL Data</h2></a>

<p>For the &quot;basic phase&quot; contest, we are providing two data sets.  The first is a demonstration data set, generated in multiple versions, containing successive realistic complications of the base measurement data.  The base demonstration data is fundamentally clean.  It contains only the deterministic spiked in data, and any residual correlations inherent in the original biological samples.  This data set is available as a control on which Visualization approaches may be prototyped, and in which all known information should be able to be observed.  Subsequent versions have been generated with increasing additional realistic biological complexity, including typical missing data, noisy data, mismeasurements and misassignments.</p>
<p>The second data set is the actual to-be-evaluated formal contest data.  The format of this data is identical to the training data, but the eQTL content is different, and it is only provided in the &quot;dirty&quot; forms.</p>
<p>Basic statistical interaction analyses are provided for both the demonstration and the formal contest data, for participants who do not wish to develop their own approaches.  These analyses provide one statistical method's measure of interactions, for two-member interactions, and for three-member interactions.  Our spiked-in data contains significantly more complex networks than these, which may only be accessible through visual synergies in visual approaches, or more sophisticated bioinformatic analyses.</p>
<p>The eQTL data catalogs genomic polymorphisms (SNPs) for 7500 genomic loci, gene expression levels for 15 genes, and an associated affected/not-affected disease state for our hypothetical spiked-in disease, hoompalitis.  The exact format of the data files is explained in text files included in the downloadable data archives, which are available in the <a href="http://biovis.net/forum">forum</a>.</p>


<a name="contest"><h2>The Challenge</h2></a>

<p>Our contest proposes a hypothetical disease, <i>hoompalitis</i>, around which we have synthesized our eQTL data.
It is known that the expression levels of several different genes affect the presence, or severity of <i>hoompalitis</i>.  Significantly abnormal levels of the gene products increases the severity of the condition, however some individuals with low and high levels appear unaffected.  Some combinations of low levels of expression of one gene product, and high levels of the others, appear to produce minimal symptoms, though not all combinations have been studied.</p>

<p>Using the data provided, identify the pattern of genome-sequence variations, and expression-levels, that predict the occurrence of <i>hoompalitis</i>.
<b>To as great an extent as possible, elucidate and explain these factors, and the pattern of interaction amongst the factors, influencing the incidence of hoompalitis</b>.</p>

<p>The factors that have been built into the data include both direct SNP effects on the genes that contain them (cis effects), SNP effects on distant genes (trans effects), networks of cis and trans-acting SNPs, and gene-gene interactions.</p>
<p>The complete pattern of interaction as spiked-in to the data, is more complex than we anticipate that any entrants will elucidate (and in fact is more complex than what is analyzable by any bioinformatic approach of which we are aware).  This complexity is, in fact, biologically realistic, and solutions that are effective with real biological data will need to take not only what can be known and shown into account, but also ideally convey the approach's limitations and the limitations of the underlying analyses.  There is a plethora of low-hanging fruit in the visualization domain for eQTL data, and we encourage contest participants to be creative in both their approaches, and in the segment of the problem they choose to pursue.</p>

<p>  
<b>A bonus question will be made available to registered entrants shortly before the contest deadline</b>.  This question will detail a specific individual within the data set, and a proposed &quot;gene therapy&quot; change to their genotype. The bonus question will be to predict the outcome for this individual if the proposed therapy were carried out, based on what you have learned about hoompalitis.
</p>

<a name="submission"><h2>Submissions</h2></a>

<p>
Participants will submit three items to document their contest entries.
</p>

<ol>
<li>A simple gridded checklist in CSV form, itemizing in general terms, which genome-sequence variants, and which genes, you predict to be relevant to the disease phenotype described in the <a href="contest.html#contest">Challenge</a> section.  A template for this file will be provided here before the submission deadline.</li>
<li>A text narrative of no more than three pages, explaining your findings, more precisely defining the network of interactions you have identified, and if possible, explaining the apparent biological &quot;meaning&quot; of the interactions. This document should also contain your answer to the <em>Bonus Question</em>.</li>

<li>A PowerPoint (or other graphical) presentation of your work, including a demonstration of your approach, and a description of how you arrived at the results reported in the checklist, and the narrative.  Video presentations are also acceptable.</li>
</ol>

<p>Please submit your contest entries via email to: <a href="mailto:contest@biovis.net">contest@biovis.net</a> no later than 11:59 pm on September 7th, 2011 (deadline extended from September 1st).</p>

<a name="judging"><h2>Judging</h2></a>
<p>
Contest results will be assayed both versus the ground truth, and versus the best understanding that Bio/Life-sciences researchers can attain using their current state-of-the-art tools. Participants will be encouraged to apply their techniques, and base their results presentation on the &quot;worst&quot; data set with which they are comfortable, and Biological-expert results will be available for comparison at each level.
</p>
<p>
Judging for the contest entries will be conducted by a panel of experts, including members from the Bio/Life-Sciences community and members from the Visualization community.  The provider of the contest data sets will participate as the final arbiter of the results' recapitulation of known ground truth, and of direct applicability to researchers in the Bio/Life Sciences domain.</p>
<p>Results will be scored independently on:
<ul>
  <li>Recapture of ground truth;</li>
  <li>Application to the problem domain;</li>
  <li>and Visualization quality.</li>
</ul>
A composite overall score and entry ranking will also be provided.</p>

<p><b>Decisions of the judging panel will be final.</b>
</p>

<a name="judges"><h2>Judges</h2></a>

Overall judging is chaired by <strong>Christoper Bartlett</strong>, The Ohio State University, USA.

<h3>Confirmatory/Biology Panel</h3>

Chaired by <strong>William Ray</strong>, The Ohio State Unversity, USA (non-voting).

	<ul>
		<li><strong>Shana R. Spindler</strong>, Editor in Chief for The NICHD Connection at the Eunice Kennedy Shriver National Institute of Child Health and Human Development, USA</li>
		<li><strong>Mark W. Logue</strong>, Assistant Research Professor, Boston University School of Medicine - Biomedical Genetics, USA</li>
		<li><strong>Wolfgang Rumpf</strong>, Product Specialist &amp; Director of Support: Rescentris Inc., USA</li>
	</ul>

<h3>Exploratory/Visualization Panel</h3>

Chaired by <strong>Raghu Machiraju</strong>, The Ohio State Unversity, USA (non-voting).

	<ul>
		<li><strong>Amitabh Varshney</strong>, University of Maryland - College Park, USA</li>
		<li><strong>Tamara Munzner</strong>, University of British Columbia, Canada</li>
		<li><strong>Ananth Grama</strong>, Purdue Unversity, USA</li>
	</ul>



<a name="awards"><h2>Awards</h2></a>

<p><strong>The following is subject to change and final decisions regarding awards are still pending.</strong></p>

<p>An award will be provided, at a minimum, for the best overall submission. Additional awards and award cash values will be announced shortly.</p>

<p>In addition to standard awards, a publication documenting the outstanding submissions, in collaboration with the lab that created the data sets for the contest (The Bartlett lab of the Battelle Center for Mathematical Medicine), will be sponsored by the contest. The Bartlett lab is also interested in co-proposing fundable projects to the National Science Foundation, National Institutes of Health, or other national or international funding opportunities.</p>



<a name="fineprint"><h2>The Fine Print</h2></a>
<p><strong>We are still printing the fine print</strong>.</p>

<p>Contest entrants agree to be bound by the following rules</p>
<ul>
  <li>No entrant or team member participating on an entry may be a current, or previous (within 3 years) collaborator of the contest organizers, judges, or data providers.  Currently excluded participants are current and prior collaborators of the Bartlett or Ray labs from the Battelle Center for Mathematical Medicine.  Additional exclusions will be made public as judges agree to serve for the contest.  If you are participating in the contest, please inform your collaborators and mentors, so that we can avoid recruiting judges that would put you in conflict!  If in doubt, ask the <a href="mailto:contest@biovis.net">contest organizers</a>.</li>
  <li>You may use any commercial or proprietary tools for either the bioinformatic analysis of what you want to visualize, or to create your visualizations.</li>
  <li>If you are providing a tool as an entry, it must run on a platform available to the judges.  We encourage statically linked Intel/Linux (Centos preferably) executables, Standalone Windows (7) executables, Standalone Mac OS X (10.6) executables, or web-based applications.  We will make every effort to make your submission run on the judge's systems, but in the end we cannot guarantee our ability to adapt the judge's systems to any particular application.</li>
  <li>At least one author or entrant from each awarded submission <b>must</b> attend the symposium to present their work.</li>
  <li>We encourage creative &quot;partial solutions&quot;. In fact, we anticipate that a complete solution is effectively intractable, because the biological reality is that <i>everything</i> interacts in biological systems in some fashion.  As a result, genius is likely to be found in identifying what not to present, and how to inform the users of those exclusions, more readily than in trying to accomplish &quot;everything&quot; in one visual approach.
</ul>

	
			</div> <!-- content -->
			
			<div class="footer">
				<p>
					BioVis 2011 is sponsored by the <a href="http://www.vgtc.org">IEEE VGTC</a> and <a href="http://www.ittc.ku.edu/chenlab/tccls">IEEE TCCLS</a>. BioVis 2011 is affiliated with the <a href="http://www.iscb.org">ISCB</a> and in-cooperation with the <a href="http://www.sigbioinformatics.org/">ACM SIG Bioinformatics</a>. To contact the organizers please send an email to <a href="mailto:contact@biovis.net">contact@biovis.net</a>.
				</p>
			</div>
		
		</div>
			
	</body>
</html>
