<html>
	<head>
		<link type="text/css" rel="stylesheet" href="css/biovis.css">
		
		<title>BioVis 2011</title>
		<script type="text/javascript">

      var _gaq = _gaq || [];
      _gaq.push(['_setAccount', 'UA-22015250-1']);
      _gaq.push(['_trackPageview']);

      (function() {
        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
      })();

    </script>
	</head>
	
	<body>
		
		<div id="frame">
			
			<div class="header">
			
				<div class="logo">
					<a href="http://www.biovis.net/2011/index.html"><img src="images/logo.png"/></a>
				</div>

				<div class="subtitle">
					<h2>1<sup>st</sup> IEEE Symposium on<br>Biological Data Visualization</h2>					
					<h3>23-24 October 2011 / Providence, RI @ IEEE VisWeek</h3>
				</div>
				
				<div class="navbar">
					<div class="menu">
					<ul>
						<li>
							<a href="http://www.biovis.net/2011/index.html">About</a>
						</li>
						<li>
							<a href="cfp.html" title="Call for Participation">CFP</a>
						</li>
						<li>
							<a href="contest.html">Contest</a>
						</li>						
						<li>
							<a href="submission.html">Submission</a>
						</li>
						<li>
							<a href="program.html">Program</a>
						</li>
						<li>
							<a href="papers_abstracts.html">Papers &amp; Abstracts</a>
						</li>						
						<li>
							<a href="registration.html">Registration</a>
						</li>
						<li>
							<a href="committee.html">Committee</a>
						</li>

					</ul>
					</div>
				</div>
				
			</div>
			
			<div class="content">
				
				<h1>Program</h1>


				<h2>Schedule</h2>
				
				
				<div class="schedule">
					<table>
						<thead>
							<tr>
								<td class="timecolumn">
								</td>
								<td class=>
									Sunday
									<span class="presenter">(Ballroom B/C)</span>
								</td>
								<td class="timecolumn">
								</td>
								<td>
									Monday
									<span class="presenter">(Ballroom B/C)</span>
								</td>							
							</tr>
						</thead>
						<tbody>
							<!-- AM session 1 -->
							<tr>
								<td class="timecolumn">
							 		<span class="time">08:30</span>
								</td>
								<td class="programcolumn">
									<span class="talk">Welcome Remarks</span>
									<span class="talk">
										<a href="program.html#keynote">Keynote</a>
										<span class="presenter">Lynda Chin</span>							
									</span>
								</td>
								<td class="timecolumn">
							 		<span class="time">08:30</span>
								</td>
								<td class="programcolumn">
									<span class="talk">
									<a href="program.html#papersession2">Paper Session 2</a> -
									Omics Data</span>
								</td>							
							</tr>
							<!-- morning break -->
							<tr class="break">
								<td class="timecolumn">
									<span class="time">10:10</span>
								</td>
								<td>
									Coffee Break
								</td>
								<td class="timecolumn">
									<span class="time">10:10</span>
								</td>
								<td>
									Coffee Break
								</td>							
							</tr>
							<!-- AM session 2 -->
							<tr>
								<td class="timecolumn">
									<span class="time">10:30</span>
								</td>
								<td class="programcolumn">
									<span class="talk">
									<a href="program.html#introsession">Primer: Molecular Biology and Bioinformatics for Data Visualization</a>
									<span class="presenter">Larry Hunter, Kun Huang</span></span>
								</td>
								<td class="timecolumn">
									<span class="time">10:30</span>
								</td>
								<td class="programcolumn">
									<span class="talk">
									<a href="program.html#papersession3">Paper Session 3</a> -
									Biological Networks, Pathways and Connectivity</span>
								</td>							
							</tr>
							<!-- lunch break -->
							<tr class="break">
								<td rowspan=2 class="timecolumn">
									<span class="time"><span class="break">12:10</span></span>
								</td>
								<td rowspan=2>
									Lunch Break
								</td>
								<td class="timecolumn">
									<span class="time">12:10</span>
								</td>
								<td  class="programcolumn">
									<span class="talk"><a href="program.html#feedbacksession">BioVis Feedback Session</a></span>
								</td>							
							</tr>
							<tr class="break">
								<td class="timecolumn">
									<span class="time">01:00</span>
								</td>
								<td>
									Lunch Break
								</td>							
							</tr>
							<!-- PM session 1 -->
							<tr>
								<td class="timecolumn">
									<span class="time">02:00</span>
								</td>
								<td class="programcolumn">
									<span class="talk">
									<a href="program.html#papersession1">Paper Session 1</a> -
									Molecular and Image Data</span>
								</td>
								<td class="timecolumn">
									<span class="time">02:00</span>
								</td>
								<td class="programcolumn">
									<span class="talk">
									<a href="program.html#papersession4">Paper Session 4</a> -
									Phylogenetic and Population Data, Morphometry</span>
								</td>							
							</tr>
							<!-- afternoon break -->
							<tr class="break">
								<td class="timecolumn">
									<span class="time">03:40</span>
								</td>
								<td>
									Coffee Break
								</td>
								<td class="timecolumn">
									<span class="time">03:40</span>
								</td>
								<td>
									Coffee Break
								</td>							
							</tr>
							<!-- PM session 2 -->
							<tr>
								<td rowspan="2" class="timecolumn">
									<span class="time">04:15</span>
								</td>
								<td class="programcolumn" rowspan="2">
									<span class="talk">
									<a href="program.html#postersession">Poster and Demo Session</a>
									<span class="presenter">04:15 - Fast Forward</span>
									<span class="presenter">04:45 - Viewing</span>
									</span>
								</td>
								<td class="timecolumn">
									<span class="time">04:15</span>
								</td>
								<td class="programcolumn">
									<span class="talk">
									<a href="program.html#challengessession">Challenges Session</a>
									<span class="presenter">04:15 - Cydney Nielsen</span>
									<span class="presenter">04:40 - Arthur Olson</span>
									<span class="presenter">05:05 - Willy Supatto</span>
									</span>
								</td>
							</tr>
							<tr>
								<td class="timecolumn">
									<span class="time">05:30</span>
								</td>
								<td  class="programcolumn">
									<span class="talk">Contest and Awards</span>							
									<span class="talk">Closing Remarks</span>
								</td>							
							</tr>
							<!-- evening break -->
							<!--
							<tr class="break">
								<td>
									<span class="time">TDB</span>
								</td>
								<td>
									Break
								</td>
								<td>
									Break
								</td>							
							</tr>
							-->
							<!-- evening session -->
							<tr>
								<td rowspan="2" class="timecolumn">
									<span class="time">06:00</span>
								</td>
								<td class="programcolumn" rowspan="2">
									<span class="talk"><a href="program.html#reception">Reception</a> (until 7:30)									
									<span class="presenter">Posters on display</span></span>
								</td>
								<td class="timecolumn">
									<span class="time">&nbsp;</span>
								</td>
								<td class="break">
									&nbsp;
								</td>
							</tr>
							<tr>
								<td class="timecolumn">
									<span class="time" rowspan="2">07:00</span>
								</td>
								<td class="programcolumn" rowspan="2">
									<span class="talk"><a href="program.html#bof">BioVis Contest BoF</a> (in Room 557, until 9:00)
									<span class="presenter">Meet the Experts &amp; Invitation</span></span>
								</td>
							</tr>
							<tr>
								<td rowspan="2" class="timecolumn">
									<span class="time">&nbsp;</span>
								</td>
								<td rowspan="2">
									&nbsp;
								</td>
							</tr>
							<tr>
								<td class="timecolumn">
								</td>
								<td>
								</td>
							</tr>
						</tbody>										
					</table>
				</div>

				<h2><a name="keynote">Keynote</a></h2>
				
				<h3>Translating the Cancer Genomes</h3>
				
				<p>
					Cancer is the phenotypic endpoint of myriad genetic and epigenetic aberrations that collectivey commandeer key cancer-relevant pathways.  Comprehensive characterization of the cancer genome holds enormous potential to (i) provide penetrating insight into the genetic bases of cancer, (ii) identify promising candidate therapeutic targets and diagnostic biomarkers, and (iii) illuminate the path toward personalized cancer medicine.  However, translating the complex cancer genomic information into biological insights that can impact in the clinic is an enormous challenge that involves multi-disciplinary collaboration from genomic, computational sciences to cancer genetics and disease biology, and clinical arenas.  Importantly, such collaboration begins with and depends on accessing and visualizing the data.  Examples will be shared to illustrate the importance and need for visualization.					
				</p>
				
				<div class="speaker">
					<div class="image">
						<img class="photo" src="images/lynda_chin_sm.jpg"/>
					</div>
					<div class="detail">
						<span class="name"><a href="http://www.lyndachinlab.org">Lynda Chin</a></span>
						<span class="affiliation">MD Anderson Cancer Center / Broad Institute of MIT and Harvard</span>
						<span class="bio">
							<p>
								Dr. Lynda Chin has joined University of Texas MD Anderson Cancer Center where she will be the chair of the first ever Department of Genomic Medicine and Scientific Director of the Institute for Applied Cancer Science.  For the past 14 years, she has been a member of the Dana-Farber Cancer Institute and Harvard Medical School community where she was Professor of Dermatology at the Harvard Medical School, member of Department of Medical Oncology at Dana-Farber Cancer Institute, and a Senior Associate Member of the Broad Institute.  Dr. Chin also served as the Scientific Director of the Belfer Institute for Applied Cancer Science at the Dana-Farber Cancer Institute, and co-led the Dana-Farber / Harvard Cancer Center's Melanoma Program and the Harvard Skin SPORE.  At the Broad Institute, Dr. Chin is PI of the TCGA Genome Data Analysis Center.  She is also a member of the Scientific Steering Committee of the International Cancer Genome Consortium. 
							</p>
							<p>
								Dr. Chin has made multiple scientific discoveries spanning the fields of transcription, telomere biology, mouse models of human cancer, oncogenomics, and personalized cancer medicine. She co-founded AVEO Pharmaceuticals in 2002, a cancer biotechnology company that emphasizes cancer biology and genetics to identify new cancer targets with tumor maintenance roles.  Most recently, Dr. Chin also founded Metamark Genetic, a cancer diagnostic company that will develop function-based prognostic determinants that can guide customized management of early-staged cancer patients including melanoma and prostate cancer. 						
							</p>
							<p>
								Dr. Chin received her M.D. from the Albert Einstein College of Medicine in 1993.  She is a board-certified dermatologist and conducted her clinical and scientific training at Columbia Presbyterian Medical Center and the Albert Einstein College of Medicine where she served as Chief Resident of Dermatology. Dr. Chin has three young children, ages 10, 9 and 7.							
							</p>
						</span>						
					</div>
				</div>
						
								
				<h2><a name="introsession">Primer: Molecular Biology and Bioinformatics for Data Visualization</a></h2>
				
				<p>
				 The speakers will provide a general overview of current practices and tools in bioinformatics and also highlight the role visualization can play. More details to follow.
				</p>
								
				<h3>Sense-Making in Molecular Biology</h3>
				
				<p>
					The advent of genome-scale molecular instrumentation has led to an explosion of biomedically relevant data, with nonstop exponential improvements in performance likely to continue for decades. We are truly at an inflection point in the history of life and medicine. However, this deluge presents a significant challenge: making sense of genome scale data is increasingly the fundamental task in molecular biology. In this brief tutorial, we will explore the different sorts of explanations that play a role in understanding life, the sources of knowledge about genes and their effects, and the process of integrating knowledge and data about mul- tiple genetic differences into a coherent story about mechanism.
				</p>
				
				<div class="speaker">
					<div class="image">
						<img class="photo" src="images/larry_hunter_sm.jpg"/>
					</div>
					<div class="detail">
						<span class="name"><a href="http://compbio.ucdenver.edu/hunter/">Lawrence Hunter</a></span>
						<span class="affiliation">University of Colorado</span>
						<span class="bio">
							<p>
							Lawrence Hunter is the Director of the Computational Bioscience Program at the University of Colorado School of Medicine, and is a Professor in the departments of Pharmacology and Computer Science (Boulder). He received his Ph.D. in computer science from Yale University in 1989, and then spent more than 10 years at the National Institutes of Health, ending as the Chief of the Molecular Statistics and Bioinformatics Section at the National Cancer Institute.
							</p>
							<p>
							He inaugurated two of the most important academic bioinformatics conferences, ISMB and PSB, and was the founding President of the International Society for Computational Biology (ISCB). He was named a Fellow of the ISCB in 2009.
							</p>
							<p>
							Larry's research interests span a wide range of areas, from cognitive science to rational drug design. His primary focus recently has been the integration of natural language processing, knowledge representation and machine learning techniques and their application to interpreting data generated by high throughput molecular biology.
							<p>
						</span>						
					</div>
				</div>
					
				<h3>Applications and Computing Challenges with Next Generation Sequencing</h3>
				
				<p>
					During the past five years,  Next Generation Sequencing (NGS) technology is changing biomedical research in a revolutionary way. For the first time, researchers can directly study the genome at single-base resolution in a relatively inexpensive manner. There have been many applications utilizing these technology including ChIP-seq (protein-DNA interaction), RNA-seq (transcriptome), SNP discovery, methylome, genome de novo sequencing and re-sequencing. The high accuracy and high throughput of the technology also leads to quick accumulation of huge amount of data. In this talk, I will give an overview on a few applications of NGS in cancer research and the challenges associated with the large data including data visualization, analysis, management, and integration. 			
				</p>
				
				
				<div class="speaker">
					<div class="image">
						<img class="photo" src="images/kun_huang_sm.jpg"/>
					</div>
					<div class="detail">
						<span class="name"><a href="http://bmi.osu.edu/~khuang/">Kun Huang</a></span>
						<span class="affiliation">The Ohio State University</span>
						<span class="bio">
							<p>
								Kun Huang received two B.S. degrees in biology and computer science from Tsinghua University, Beijing, China, in 1996. In addition, he received the M.S. degree in physiology in 1998, the M.S. degree in electrical engineering in 2000, the M.S. degree in mathematics in 2002, and the Ph.D. degree in electrical and computer engineering in 2004, all from the University of Illinois at Urbana-Champaign. Currently he is an Associate Professor in the Department of Biomedical Informatics, The Ohio State University (OSU), Columbus, Ohio, USA and the Co-Director of the OSU Comprehensive Cancer Center Biomedical Informatics Shared Resources. His research interests include computational biology, bioinformatics, biomedical imaging analysis, computer vision, and machine learning.
							</p>								
						</span>						
					</div>
				</div>
					
				
				<h2><a name="papersession1">Paper Session 1 - Molecular and Image Data</a></h2>
				
				<p>
					<strong>Session Chair:</strong> Chris Johnson, University of Utah, USA
				</p>
				
				<p>
<div class="presentation short">
	<!-- Paper #136 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/136.html"><img src="images/papers/50/136.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/136.html">Quick2Insight: A User-Friendly Framework for Interactive Rendering of Biological Image Volumes</a></span>
	<span class="author">Yanling Liu, Curtis Lisle, Jack Collins</span>
</div>

<div class="presentation short">
	<!-- Paper #158 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/158.html"><img src="images/papers/50/158.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/158.html">Modeling and Visualization of Receptor Clustering on the Cellular Membrane</a></span>
	<span class="author">Martin Falk, Markus Daub, Guido Schneider, Thomas Ertl</span>
</div>

<div class="presentation short">
	<!-- Paper #130 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/130.html"><img src="images/papers/50/130.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/130.html">Parallel Contour-Buildup Algorithm for the Molecular Surface</a></span>
	<span class="author">Michael Krone, Sebastian Grottel, Thomas Ertl</span>
</div>

<div class="presentation short">
	<!-- Paper #129 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/129.html"><img src="images/papers/50/129.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/129.html">Fiber Stippling: An Illustrative Rendering for Probabilistic Diffusion Tractography</a></span>
	<span class="author">Mathias Goldau, Alexander Wiebel, Nico Stephan Gorbach, Corina Melzer, Mario Hlawitschka, Gerik Scheuermann, Marc Tittgemeyer</span>
</div>

<div class="presentation short">
	<!-- Paper #133 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/133.html"><img src="images/papers/50/133.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/133.html">Visualization of Anisotropic Contact Potentials within Protein Structures</a></span>
	<span class="author">Corinna Vehlow, Bernhard Preim, Michael Lappe</span>
</div>

<div class="presentation short">
	<!-- Abstract #119 -->		
	<span class="thumbnail"><a href="papers_abstracts/abstracts/119.html"><img src="images/abstracts/50/119.png"/></a></span>
	<span class="title">Abstract - <a href="papers_abstracts/abstracts/119.html">Neural Process Reconstruction from Sparse User Scribbles</a></span>
	<span class="author">Mike Roberts, Won-Ki Jeong, Amelio Vazquez-Reina, Markus Unger, Horst Bischof, Jeff Lichtman, Hanspeter Pfister</span>
</div>
				</p>
				


				<h2><a name="papersession2">Paper Session 2 - Omics Data</a></h2>

				<p>
					<strong>Session Chair:</strong> Matt Hibbs, The Jackson Laboratory, USA
				</p>

				<p>
<div class="presentation short">
	<!-- Paper #111 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/111.html"><img src="images/papers/50/111.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/111.html">The Galaxy Track Browser: Transforming the Genome Browser from Visualization Tool to Analysis Tool</a></span>
	<span class="author">Jeremy Goecks, Kanwei Li, Dave Clements, James Taylor</span>
</div>
					
<div class="presentation short">
	<!-- Paper #140 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/140.html"><img src="images/papers/50/140.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/140.html">Visual Analysis of Next-Generation Sequencing Data to Detect Overlapping Genes in Bacterial Genomes</a></span>
	<span class="author">Svenja Simon, Daniela Oelke, Richard Landstorfer, Klaus Neuhaus, Daniel A. Keim</span>
</div>

<div class="presentation short">
	<!-- Paper #113 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/113.html"><img src="images/papers/50/113.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/113.html">TIALA - Time Series Alignment Analysis</a></span>
	<span class="author">G&#252;nter J&#228;ger, Florian Battke, Kay Nieselt</span>
</div>

<div class="presentation short">
	<!-- Paper #150 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/150.html"><img src="images/papers/50/150.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/150.html">iHAT: interactive Hierarchical Aggregation Table</a></span>
	<span class="author">Corinna Vehlow, Julian Heinrich, Florian Battke, Daniel Weiskopf, Kay Nieselt</span>
</div>

<div class="presentation short">
	<!-- Paper #106 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/106.html"><img src="images/papers/50/106.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/106.html">A Visual Analysis System for Metabolomics Data</a></span>
	<span class="author">Philip Livengood, Ross Maciejewski, Wei Chen, David Ebert</span>
</div>

<div class="presentation short">
	<!-- Abstract #135 -->		
	<span class="thumbnail"><a href="papers_abstracts/abstracts/135.html"><img src="images/abstracts/50/135.png"/></a></span>
	<span class="title">Abstract - <a href="papers_abstracts/abstracts/135.html">G-nome Surfer: a Tabletop Interface for Collaborative Exploration of Genomic Visualization</a></span>
	<span class="author">Orit Shaer, Megan Strait, Consuelo Valdes, Taili Feng, Mikey Lintz, Heidi Wang</span>
</div>
				</p>
				
				
				<h2><a name="papersession3">Paper Session 3 - Biological Networks, Pathways and Connectivity</a></h2>
				
				<p>
					<strong>Session Chair:</strong> Guy Melan&ccedil;on, University of Bordeaux, France
				</p>
				

				<p>				
<div class="presentation short">
	<!-- Paper #148 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/148.html"><img src="images/papers/50/148.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/148.html">Metrics for Comparing Explicit Representations of Interconnected Biological Networks</a></span>
	<span class="author">David Mayerich, Chris Bjornsson, Jonothan Taylor, Badrinath Roysam</span>
</div>

<div class="presentation short">
	<!-- Paper #104 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/104.html"><img src="images/papers/50/104.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/104.html">GenAMap: Visualization Strategies for Structured Association Mapping</a></span>
	<span class="author">Ross Curtis, Peter Kinnaird, Eric Xing</span>
</div>

<div class="presentation short">
	<!-- Paper #160 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/160.html"><img src="images/papers/50/160.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/160.html">HiTSEE: A Visualization Tool for Hit Selection and Analysis in High-Throughput Screening Experiments</a></span>
	<span class="author">Enrico Bertini, Hendrik Strobelt, Joachim Braun, Oliver Deussen, Ulrich Groth, Thomas U. Mayer, Dorit Merhof</span>
</div>

<div class="presentation short">
	<!-- Paper #149 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/149.html"><img src="images/papers/50/149.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/149.html">RuleBender: Integrated Visualization for Biochemical Rule-Based Modeling</a></span>
	<span class="author">Adam M. Smith, Wen Xu, Yao Sun, James R. Faeder, G. Elisabeta Marai</span>
</div>

<div class="presentation short">
	<!-- Paper #118 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/118.html"><img src="images/papers/50/118.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/118.html">MDMap: A System for Data-driven Layout and Exploration of Molecular Dynamics Simulations</a></span>
	<span class="author">Robert Patro, Cheuk Yiu Ip, Sujal Bista, Samuel Cho, D. Thirumalai, Amitabh Varshney</span>
</div>

<div class="presentation short">
	<!-- Abstract #140 -->		
	<span class="thumbnail"><a href="papers_abstracts/abstracts/140.html"><img src="images/abstracts/50/140.png"/></a></span>
	<span class="title">Abstract - <a href="papers_abstracts/abstracts/140.html">InBox: In-situ Multiple-Selection and Multiple-View Exploration of Diffusion tensor MRI Visualization</a></span>
	<span class="author">Haipeng Cai, Jian Chen, Alexander P. Auchus, Stephen Correia, David H. Laidlaw</span>
</div>
				</p>
				
				
				<h2><a name="papersession4">Paper Session 4 - Phylogenetic and Population Data, Morphometry</a></h2>
				
				<p>
					<strong>Session Chair:</strong> David Duke, University of Leeds, UK
				</p>
				


<div class="presentation short">
	<!-- Paper #124 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/124.html"><img src="images/papers/50/124.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/124.html">Evaluating the VIPER pedigree visualisation: detecting inheritance inconsistencies in genotyped pedigrees.</a></span>
	<span class="author">Trevor Paterson, Martin Graham, Jessie Kennedy, Andy Law</span>
</div>

<div class="presentation short">
	<!-- Paper #134 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/134.html"><img src="images/papers/50/134.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/134.html">Visual Exploration of Microbial Populations</a></span>
	<span class="author">Sara Johansson Fernstad, Jimmy Johansson, Suzi Adams, Jane Shaw, David Taylor</span>
</div>

<div class="presentation short">
	<!-- Paper #144 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/144.html"><img src="images/papers/50/144.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/144.html">Visualizing Virus Population Variability From Next Generation Sequencing Data</a></span>
	<span class="author">Michael Correll, Subhadip Ghosh, David O'Connor, Michael Gleicher</span>
</div>

<div class="presentation short">
	<!-- Paper #168 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/168.html"><img src="images/papers/50/168.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/168.html">EVEVis: A Multi-Scale Visualization System for Dense Evolutionary Data</a></span>
	<span class="author">Robert Miller, Vadim Mozhayskiy, Ilias Tagkopoulos, Kwan-Liu Ma</span>
</div>
	
<div class="presentation short">
	<!-- Paper #151 -->	
	<span class="thumbnail"><a href="papers_abstracts/papers/151.html"><img src="images/papers/50/151.png"/></a></span>
	<span class="title"><a href="papers_abstracts/papers/151.html">Semantically Steered Visual Analysis of Highly Detailed Morphometric Shape Spaces</a></span>
	<span class="author">Max Hermann, Anja C. Schunke, Reinhard Klein</span>
</div>

<div class="presentation short">
	<!-- Abstract #117 -->		
	<span class="thumbnail"><a href="papers_abstracts/abstracts/117.html"><img src="images/abstracts/50/117.png"/></a></span>
	<span class="title">Abstract - <a href="papers_abstracts/abstracts/117.html">Interactive visualization of multiscale biomedical data: an integrated approach</a></span>
	<span class="author">Debora Testi, Gordon Clapworthy, Stephen Aylward, Alejandro Frangi, Richard Christie</span>
</div>
				</p>
				

				<h2><a name="postersession">Poster and Demo Session</a></h2>
				
				<p>
					Authors of accepted abstracts will be presenting their posters and interactive demos during this session. The session will be opened with a brief Fast Forward that will include all accepted abstracts. During the Fast Forward, one author from each abstract will have 30 seconds to present a slide summarizing their work and promote their poster/demo to the symposium attendees.				
				</p>
				<p>	
					 The list of accepted abstracts is available in the <a href="papers_abstracts.html#abstracts">Papers &amp; Abstracts</a> section.</em>
				</p>
				
				
				<h2><a name="reception">Reception</a></h2>

				<p>
					This will be a joint event with the <a href="http://ldav.org">Large-Scale Data Analysis and Visualization Symposium</a> and will be open to all VisWeek attendees. BioVis posters and demos will continue to be on display during the reception.
				</p>

				<p>
					<i>This event is made possible by the generous financial support provided by <a href="http://www.autodesk.com">Autodesk</a> and <a href="http://www.agilent.com">Agilent Technologies</a></i>.
				</p>
				
				
				<h2><a name="feedbacksession">BioVis Feedback Session</a></h2>
				
				<p>
					This session will provide an opportunity for all interested attendees to discuss the BioVis 2011 meeting and plans for future meetings with members of the <a href="committee.html">Organizing Committee</a>. Everybody is welcome!
				</p>
				
							
				<h2><a name="challengessession">Challenges Session</a></h2>
				
				<p>
					We have selected a range of speakers that, together with the keynote speaker, will cover developments in a broad range of active research topics in modern biological data visualization, from genes to proteins to organisms to populations. The speakers include developers of visualization tools widely used in biology, as well as active users who apply visualization methods to discover new biological knowledge.
				</p>

				<h3>Visualizing next-generation genomic data</h3>
				<p>
					As our ability to generate sequencing data continues to increase, data analysis is replacing data generation as the rate-limiting step in genomics studies. This talk will highlight some of the key visualization approaches that currently facilitate genomic analysis tasks by enabling researchers to explore, interpret and manipulate their data. I will emphasize the challenges ahead, particularly in terms of data representation and the integration of visual and computational analysis techniques.
				</p>

				<div class="speaker">
					<div class="image">
						<img class="photo" src="images/cydney_nielsen_sm.jpg"/>
					</div>
					<div class="detail">
						<span class="name"><a href="http://www.cydney.org">Cydney Nielsen</a></span>
						<span class="affiliation">BC Cancer Agency Genome Sciences Centre</span>
						<span class="bio">
							<p>
								Cydney Nielsen is a Postdoctoral Fellow at the BC Cancer Agency's Genome Sciences Centre. She completed her PhD in Computational Biology at MIT where her work in Chris Burge's lab focused on understanding mammalian post-transcriptional gene regulation. Motivated by the analysis challenges posed by high-throughput sequencing technologies, her current research exploits synergies between human visual reasoning and machine computation in order to address pressing questions in genomics. Her work on a genome assembly visualization tool, ABySS-Explorer, won a Best Paper award at the 2009 IEEE Information Visualization Conference.
							</p>
						</span>						
					</div>
				</div>

				<h3>Visualization of biomolecular assemblies and environments</h3>
				
				<p>
					Gaining insight into biological processes at a molecular scale requires visualizing biological assemblies using a synthesis of both abstract &amp; 3D visualization. This talk will highlight currently emerging methods used for exploring molecular recognition and self-assembly using hybrid user interfaces that combines 3D solid printing and augmented reality environments, as well as methods for fast, integrated, and large-scale macromolecular visualization.
				</p>

				<div class="speaker">
					<div class="image">
						<img class="photo" src="images/arthur_olson_sm.jpg"/>
					</div>
					<div class="detail">
						<span class="name"><a href="http://mgl.scripps.edu">Arthur Olson</a></span>
						<span class="affiliation">The Scripps Research Institute</span>
						<span class="bio">
							<p>
								Arthur Olson is the Anderson Research Chair Professor in the Department of Molecular Biology at The Scripps Research Institute and founder and director of its Molecular Graphics Laboratory. He received his B.S. (Honors) in Chemistry from the University of Michigan and his Ph.D. in Physical Chemistry from the University of California, Berkeley.  He was a Damon Runyon Postdoctoral Fellow at Harvard University, working on the first atomic resolution structure of an intact viral capsid.
							</p>
							<p>
								He is a pioneer in the analysis and visualization of biological assemblies spanning length scales from microns to angstroms. His laboratory has developed, applied and distributed a broad range of molecular modeling and visualization software tools over the past 25 years, including AutoDock, which is the worlds most highly cited ligand protein docking code. He started the first Internet distributed biomedical computing project, FightAIDS@Home, which is now running on over a million computers worldwide, and which was honored by resolution in the California State Legislature. His latest work in molecular graphics focuses on the development of novel and intuitive human interfaces for research and education in structural molecular biology utilizing solid printing and augmented reality technologies. Olson's visualizations and animations have reached a broad audience through public venues such as the Disney EPCOT center, PBS television, and a number of art and science museum exhibits around the world.
							</p>
						</span>						
					</div>
				</div>
				
				<h3>Visualizing embryonic development</h3>
				
				<p>
					Advances in microscopy now allow collection of 4D image data of embryonic development at unprecedented time and spatial resolution. This talk shows how this image data is being used to reveal new insights into developmental patterns, and highlights the need for more specifically-tailored visualization tools to manage these multidimensional data. The talk also discusses the challenges of combining data from multiple experiments, integrating raw imaging data and performing multidimensional analysis.
				</p>
				
			
				<div class="speaker">
					<div class="image">
						<img class="photo" src="images/willy_supatto_sm.jpg"/>
					</div>
					<div class="detail">
						<span class="name"><a href="http://willy.supatto.perso.sfr.fr">Willy Supatto</a></span>
						<span class="title"></span>
						<span class="affiliation">CNRS - Ecole Polytechnique</span>
						<span class="bio">
							<p>
								Willy Supatto is a tenured CNRS researcher at the Laboratory for Optics and Biosciences at Ecole Polytechnique (Palaiseau, France) since 2011. He received a M.Sc. degree in Physics and Chemistry from ESPCI ParisTech (Paris, France) and a M.Sc. degree in Biophysics from Paris University. He completed his Ph.D. in Biophysics at Institut Curie in Paris where his work in Emmanuel Farge's lab focused on developing new imaging and optical manipulation strategies to investigate biomechanics during embryonic morphogenesis. In 2006, he joined the Biological Imaging Center at the California Institute of Technology (Caltech, Pasadena, USA) to pursue postdoctoral studies with Pr. Scott E. Fraser within the Center of Excellence in Genomics Science. He developed microscopy, optical manipulation and image analysis tools to study symmetry breaking, fluid flow dynamics and collective cell migration in living embryos. He joined the Ecole Polytechnique in 2011 as a CNRS researcher to pursue pioneering investigaion of embryonic development using quantitative imaging.
							</p>								
						</span>						
					</div>
				</div>
				
				<h2><a name="bof">BioVis Contest BoF - Meet the Experts &amp; Invitation</a></h2>
				
				<p>
					The foremost purpose of applying visualization to biology is to improve biological experts' access to information, and the <strong>BioVis Contest</strong> exists to help visualization experts develop tools that are better suited for that purpose. Come meet the biological domain experts who created the data for this year's contest, and talk with real users of eQTL data who desperately need better tools. Learn more about the domain, and the end users' real needs. 
				</p>
				
				<p>
					Find out what's brewing for Round 2 of the eQTL data analysis challenge, and help plan the theme for the next BioVis Round 1 challenge.				
				</p>
				
				
			</div>
						
			
			<div class="footer">
				<p>
					BioVis 2011 is sponsored by the <a href="http://www.vgtc.org">IEEE VGTC</a> and <a href="http://www.ittc.ku.edu/chenlab/tccls">IEEE TCCLS</a>. BioVis 2011 is affiliated with the <a href="http://www.iscb.org">ISCB</a> and in-cooperation with the <a href="http://www.sigbioinformatics.org/">ACM SIG Bioinformatics</a>. To contact the organizers please send an email to <a href="mailto:contact@biovis.net">contact@biovis.net</a>.
				</p>
			</div>
		
		</div>
			
	</body>
</html>
