<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN"
  "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" version="XHTML+RDFa 1.0" dir="ltr"
  xmlns:content="http://purl.org/rss/1.0/modules/content/"
  xmlns:dc="http://purl.org/dc/terms/"
  xmlns:foaf="http://xmlns.com/foaf/0.1/"
  xmlns:og="http://ogp.me/ns#"
  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
  xmlns:sioc="http://rdfs.org/sioc/ns#"
  xmlns:sioct="http://rdfs.org/sioc/types#"
  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
  xmlns:xsd="http://www.w3.org/2001/XMLSchema#">

<head profile="http://www.w3.org/1999/xhtml/vocab">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta about="/year/2013/info/speakers" property="sioc:num_replies" content="0" datatype="xsd:integer" />
<link rel="shortcut icon" href="../sites/all/themes/biovis/favicon.ico" type="image/vnd.microsoft.icon" />
<meta content="Speakers" about="/year/2013/info/speakers" property="dc:title" />
<link rel="shortlink" href="525.html" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="../year/2013/info/speakers.html" />
  <title>Speakers | BioVis</title>
  <style type="text/css" media="all">@import url("http://www.biovis.net/modules/system/system.base.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.menus.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.messages.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.theme.css?njdarp");</style>
<style type="text/css" media="all">@import url("http://www.biovis.net/modules/comment/comment.css?njdarp");
@import url("../sites/all/modules/date/date_api/date.css@njdarp.css");
@import url("../sites/all/modules/date/date_popup/themes/datepicker.1.7.css@njdarp.css");
@import url("http://www.biovis.net/modules/field/theme/field.css?njdarp");
@import url("http://www.biovis.net/modules/node/node.css?njdarp");
@import url("http://www.biovis.net/modules/search/search.css?njdarp");
@import url("http://www.biovis.net/modules/user/user.css?njdarp");
@import url("../sites/all/modules/views/css/views.css@njdarp.css");</style>
<style type="text/css" media="all">@import url("../sites/all/modules/ckeditor/ckeditor.css@njdarp.css");
@import url("../sites/all/modules/ctools/css/ctools.css@njdarp.css");</style>
<style type="text/css" media="all">@import url("http://www.biovis.net/themes/bartik/css/layout.css?njdarp");
@import url("http://www.biovis.net/themes/bartik/css/style.css?njdarp");
@import url("../sites/default/files/color/biovis-8495a6ec/colors.css@njdarp.css");
@import url("../sites/all/themes/biovis/css/biovis.css@njdarp.css");</style>
<style type="text/css" media="print">@import url("http://www.biovis.net/themes/bartik/css/print.css?njdarp");</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="http://www.biovis.net/sites/all/themes/biovis/css/ie.css?njdarp" media="all" />
<![endif]-->

<!--[if IE 6]>
<link type="text/css" rel="stylesheet" href="http://www.biovis.net/sites/all/themes/biovis/css/ie6.css?njdarp" media="all" />
<![endif]-->
  <script type="text/javascript" src="http://www.biovis.net/misc/jquery.js?v=1.4.4"></script>
<script type="text/javascript" src="http://www.biovis.net/misc/jquery.once.js?v=1.2"></script>
<script type="text/javascript" src="http://www.biovis.net/misc/drupal.js?njdarp"></script>
<script type="text/javascript" src="../sites/all/modules/google_analytics/googleanalytics.js@njdarp"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _gaq = _gaq || [];_gaq.push(["_setAccount", "UA-22015250-1"]);_gaq.push(["_trackPageview"]);(function() {var ga = document.createElement("script");ga.type = "text/javascript";ga.async = true;ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";var s = document.getElementsByTagName("script")[0];s.parentNode.insertBefore(ga, s);})();
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"biovis","theme_token":"mal3JQzqMMQZ98pNa_wH5gTQCMiZkq_08wEihhkOC1M","js":{"misc\/jquery.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/google_analytics\/googleanalytics.js":1,"0":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/date\/date_api\/date.css":1,"sites\/all\/modules\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/views\/css\/views.css":1,"sites\/all\/modules\/ckeditor\/ckeditor.css":1,"sites\/all\/modules\/ctools\/css\/ctools.css":1,"themes\/bartik\/css\/layout.css":1,"themes\/bartik\/css\/style.css":1,"sites\/all\/themes\/biovis\/css\/colors.css":1,"sites\/all\/themes\/biovis\/css\/biovis.css":1,"themes\/bartik\/css\/print.css":1,"sites\/all\/themes\/biovis\/css\/ie.css":1,"sites\/all\/themes\/biovis\/css\/ie6.css":1}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls|xml|z|zip"}});
//--><!]]>
</script>
  <base href="" />
</head>
<body class="html not-front not-logged-in one-sidebar sidebar-second page-node page-node- page-node-525 node-type-conference-info" >
  <div id="skip-link">
    <a href="../index.html#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
    <div id="page-wrapper"><div id="page">

  <div id="header" class="without-secondary-menu"><div class="section clearfix">

          <a href="../year/2015/about.html" title="Home" rel="home" id="logo">
        <img src="../sites/default/files/logo_3.png" alt="Home" />
      </a>
    
          <div id="name-and-slogan">

                              <div id="site-name">
              <strong>
                <span class="conference-name">5th Symposium on Biological Data Visualization</span>
              </strong>
			  <br />
			    <span class="conference-info">10- 11 July // Dublin, Ireland @ ISMB/ECCB 2015</span>
            </div>
                  
        
      </div> <!-- /#name-and-slogan -->
				  <div id="header-rotation">
		    <img src="../sites/all/themes/biovis/header_rotation/2015/134.png" alt=""/>		  </div>
		    
    
          <div id="main-menu" class="navigation">
	  <h2 class="element-invisible">Main menu</h2><ul id="main-menu-links" class="links clearfix"><li class="menu-592 first"><a href="../year/2015/about.html" title="">About</a></li>
<li class="menu-610"><a href="../year/2015/news.html" title="News">News</a></li>
<li class="menu-596"><a href="../year/2015/cfp.html" title="">CFP</a></li>
<li class="menu-597"><a href="../year/2015/program.html" title="">Program</a></li>
<li class="menu-598"><a href="../year/2015/papers.html" title="">Papers</a></li>
<li class="menu-599"><a href="../year/2015/posters/index.html" title="">Posters</a></li>
<li class="menu-601"><a href="../year/2015/design/update.html" title="">Design Contest</a></li>
<li class="menu-600"><a href="../year/2015/info/overview.html" title="">Data Contest</a></li>
<li class="menu-603"><a href="../year/2015/registration.html" title="Registration page">Registration</a></li>
<li class="menu-594"><a href="../year/2015/info/2015-committee.html" title="">Committee</a></li>
<li class="menu-595 last"><a href="../conference-archive.html" title="">Archive</a></li>
</ul>      </div> <!-- /#main-menu -->
    

    
  </div></div> <!-- /.section, /#header -->
          <div id="sub-menu">
          <div class="region region-sub-menu">
    <div id="block-menu-block-2" class="block block-menu-block">

    
  <div class="content">
    <div class="menu-block-wrapper menu-block-2 menu-name-menu-main-menu-2013 parent-mlid-0 menu-level-2">
  <ul class="menu clearfix"><li class="first leaf menu-mlid-557"><a href="../year/2013/info/schedule.html" title="">Schedule</a></li>
<li class="last leaf active-trail active menu-mlid-558"><a href="../year/2013/info/speakers.html" title="" class="active-trail active">Speakers</a></li>
</ul></div>
  </div>
</div>
  </div>
      </div> <!-- /.section, /#sub-menu -->
    
  
  
  <div id="main-wrapper" class="clearfix"><div id="main" class="clearfix">

    
    
    <div id="content" class="column"><div class="section">
            <a id="main-content"></a>
                    <h1 class="title" id="page-title">
          Speakers        </h1>
                          <div class="tabs">
                  </div>
                          <div class="region region-content">
    <div id="block-system-main" class="block block-system">

    
  <div class="content">
    <div id="node-525" class="node node-conference-info node-full clearfix" about="/year/2013/info/speakers" typeof="sioc:Item foaf:Document">

      
  
  <div class="content clearfix">
    <div class="field field-name-body field-type-text-with-summary field-label-hidden"><div class="field-items"><div class="field-item even" property="content:encoded"><h2>
	Keynote</h2>
<h3>
	<a name="keynote">Scientific Publishing in a Technological Age</a></h3>
<div>
	The scientific publishing landscape is changing rapidly and publishers, authors and readers are working hard to adapt. Advances in research technologies are generating increasing amounts of data that require commensurate advances in accessibility, analysis and visualization. Although digital technologies provide possible solutions to allow publishers to fully participate in this revolution and add new capabilities, it has proven challenging to adapt the traditional publishing model to take full advantage of new opportunities. I will discuss these challenges and opportunities and what publishers are doing to meet them from the point of view of a chief editor at Nature Publishing Group. Particular emphasis will be given to data handling and visualization, the role of computer scientists and the challenges in bridging the distinct biology and computer science cultures.</div>
<table border="0" cellpadding="7" style="border-width: 0px; border-color: gray; border-style: solid;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" style="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Christof Koch" id="mce-757" src="../sites/default/files/daniel_evanko_0.jpg" style="display: block; margin-left: auto; margin-right: auto; width: 150px; height: 170px;" title="Christof Koch" /></td>
			<td>
				<p><a href="http://www.nature.com/nmeth/about/about_eds/index.html#evanko">Daniel Evanko, Ph.D.</a><br />
					Chief Editor, Nature Methods</p>
				<hr />
				<p>Daniel Evanko graduated from Northwestern University with a B.S. in Biomedical Engineering. He obtained a Ph.D. in Molecular Pharmacology and Structural Biology and performed post-doctoral work on G protein targeting and regulation at Thomas Jefferson University. As a Research Associate at the University of Pennsylvania Dr. Evanko shifted his scientific focus to fluorescence imaging technologies for studying astrocyte-neuron interactions. He joined the staff of Nature Methods in August of 2004 as an Assistant Editor handling manuscripts and the first issue was published in October. After serving as an Associate and then Senior Editor he became Chief Editor in 2008. While at Nature Methods Dr. Evanko has interviewed dozens of scientists, edited hundreds of manuscripts, written over 100 articles, created and edited a popular column of practical advice on visualizing scientific data, and in 2013 started a new column on statistics. He has been involved in numerous internal initiatives, including ones related to visualization and statistics.&nbsp;</p>
			</td>
		</tr>
	</tbody>
</table>
<h2>
	Challenges Session</h2>
<p>We have selected a range of speakers that, together with the keynote speaker, will cover developments in a broad range of active research topics in modern biological data visualization, from genes to proteins to organisms to populations. The speakers include developers of visualization tools widely used in biology, as well as active users who apply visualization methods to discover new biological knowledge.</p>
<h3>
	<a name="challenge1">Engaging today&rsquo;s genomics resources</a></h3>
<p>Advances in next-generation sequencing platforms have reshaped the landscape of genomic and epigenomic research. Large consortiums such as the ENCyclopedia of DNA Elements (ENCODE), the Roadmap Epigenomics project and The Cancer Genome Atlas (TCGA) have generated tens of thousands of sequencing-based genome-wide datasets, creating a reference and resource for the scientific community. Small groups of researchers are now able to rapidly obtain huge volumes of genomic data and are in great need of putting their own data in the context of the consortium data for comparison. These data are often accompanied by rich metadata describing the sample and experiment, critical for their interpretation. Fast evolving technologies, such as those interrogating long-range chromatin interactions, are generating new types of data in similarly large volumes. Visualizing, navigating, and interpreting such data in a meaningful way is a daunting challenge. In this presentation I will discuss the present challenges in genomic data visualization and the rapid evolution of Genome Browsers.</p>
<table border="0" cellpadding="7" style="border-width: 0px; border-color: gray; border-style: solid;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" style="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Inna Dubchak" id="mce-757" src="../sites/default/files/TingWang.png" style="display: block; margin-left: auto; margin-right: auto; width: 150px; height: 197px;" title="Inna Dubchak" /></td>
			<td>
				<p><a href="http://wang.wustl.edu/">Ting Wang, Ph.D.</a><br />
					Assistant Professor at the Genetics Department and Center for Genome Sciences, Washington University in St. Louis</p>
				<hr />
				<p>Ting Wang is an assistant professor at the Genetics Department and Center for Genome Sciences at Washington University in St. Louis. He has a PhD in Computational Biology. His research focuses on understanding genetic and epigenetic impact of transposable element on human regulatory networks and their role in human diseases. He develops algorithms for identifying regulatory motifs, and analytical and visualization methodologies to integrate genomic and epigenomic data. He is a co-inventor of the UCSC Cancer Genomics Browser, inventor of the Wash U Epigenome Browser, and a co-investigator of the Epigenome Roadmap Mapping Centers.</p>
			</td>
		</tr>
	</tbody>
</table>
<h3>
	<a name="challenge2">Tools for Interactive Visualization and Analysis of Complex Biological Structures</a></h3>
<p>As experimental methods to determine the detailed structures of molecules and molecular assemblies continue to rapidly advance, and as databases of modeled macromolecular complexes also swiftly grow in both size and complexity, the challenges of easily visualizing, analyzing, and comparing these structures become increasing difficult. &nbsp;Particularly onerous are models that encompass scales that range from the atomic to cellular and data that is dynamical in nature. &nbsp;Yet the ambitious goal of creating an accurate spatiotemporal model of the cell and all of its molecular processes, with maximal accuracy, resolution, and completeness, is nevertheless crucial to a thorough understanding of biological function and the molecular basis of disease. &nbsp;Because of the complexity of this data, interactive 3D visualization tools greatly facilitate analysis tasks by enabling researchers to explore, interpret, and manipulate their data. &nbsp;This talk will highlight the challenges associated with visualization of macromolecules at multiple spatial resolutions and illustrate some of the existing tools used to address this problem domain. &nbsp;Also to be discussed are the challenges that tool builders must face if their tools are to be adopted by the larger scientific community.</p>
<table border="0" cellpadding="7" style="border-width: 0px; border-color: gray; border-style: solid;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" style="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Valerie Daggett" id="mce-757" src="../sites/default/files/TomFerrin.png" style="display: block; margin-left: auto; margin-right: auto; width: 150px; height: 180px;" title="Valerie Daggett, PhD" /></td>
			<td>
				<p><a href="http://www.cgl.ucsf.edu/home/tef">Thomas Ferrin, Ph.D.</a><br />
					Professor, Department of Pharmaceutical Chemistry and Department of Bioengineering and Therapeutic Sciences<br />
					Director, Resource for Biocomputing, Visualization, and Informatics,&nbsp;University of California, San Francisco</p>
				<hr />
				<p>Tom originally hails from Minneapolis, where he first became interested in computing while writing codes for a Control Data Corp. model 6600 computer, the first machine to be called a &ldquo;supercomputer.&rdquo; &nbsp;In the early 1970&rsquo;s, while working as a bioengineer, he became enamored with the application of minicomputers such as the Digital Equipment Corp. PDP-8 and Data General Nova for real-time control and analysis of scientific experiments. &nbsp;As the data collected from these experiments became larger and more complex, he turned to interactive computer graphics as a means of displaying and analyzing the data, and ultimately gaining insight into the underlying biophysical processes at play. &nbsp;Thus began a 40+ year career at the University of California, San Francisco (UCSF) focused on developing tools for the interactive visualization and analysis of molecular structure at scales ranging from the atomic to the supramolecular. &nbsp;In addition to tool development, Tom&rsquo;s lab has a long history of documenting, distributing, and providing training for these tools. &nbsp;Their UCSF Chimera package is used worldwide and has been cited in more than 5,400 journal articles. &nbsp;In recent years his research group has also contributed tools that emphasize integrating diverse types of biomolecular data, including atomic-resolution coordinates, density maps, sequences, annotations, and network representations of protein similarity, binding interactions, and biological pathways.</p>
			</td>
		</tr>
	</tbody>
</table>
<p>&nbsp;</p>
</div></div></div>  </div>

  
  
</div>
  </div>
</div>
  </div>
      
    </div></div> <!-- /.section, /#content -->

          <div id="sidebar-second" class="column sidebar"><div class="section">
          <div class="region region-sidebar-second">
    <div id="block-block-11" class="block block-block">

    <h2>Follow us on Twitter</h2>
  
  <div class="content">
    <p><a href="http://twitter.com/#!/search/realtime/%23biovis">@biovis_net / #biovis</a></p>
  </div>
</div>
<div id="block-block-14" class="block block-block">

    
  <div class="content">
    <p>BioVis 2013 is an official symposium of</p>
<p><a href="http://www.ieee.org"><img alt="" src="../sites/default/files/ieee_logo_sm.png" style="width: 170px; height: 50px; " /></a><br />
	<a href="http://www.computer.org"><img alt="" src="../sites/default/files/ieee_cs_logo_sm.png" style="width: 66px; height: 40px; " /></a>&nbsp;<a href="http://www.vgtc.org"><img alt="" src="../sites/default/files/ieee_vgtc_logo_sm.png" style="width: 70px; height: 40px; " /></a></p>
<p>BioVis 2013 is affiliated with</p>
<p><a href="http://www.iscb.org"><img alt="" src="../sites/default/files/iscb_logo_sm.png" style="width: 146px; height: 50px; " /></a><br />
	Join ISCB and get a 20% <a href="http://www.iscb.org/iscb-membership-benefits">member discount</a> at BioVis&#39;13.</p>
<p>&nbsp;</p>
<p>BioVis 2013 Supportes</p>
<p>Silver:</p>
<p><a href="http://www.agilent.com"><img alt="Agilent" src="../sites/default/files/4c_CorporateSig-noHL_0.jpg" style="width: 146px; height: 28px;" /></a></p>
<p>Bronze:</p>
<p><img alt="" src="../sites/default/files/autodesk_logo_sm.png" style="width: 146px; height: 30px;" /></p>
<p><a href="http://www.mathmed.org/"><img alt="Agilent" src="../sites/default/files/bcmm_logo_sm.png" style="width: 146px; height: 54px;" /></a></p>
<p>&nbsp;</p>
  </div>
</div>
<div id="block-block-12" class="block block-block">

    <h2>Deadlines</h2>
  
  <div class="content">
    <p><strong>Paper Deadline</strong><br />
	- Abstract:<br />
	&nbsp; April 30, 2013<br />
	- Full Paper:<br />
	&nbsp; May 7, 2013<br />
	<strong>Poster Deadline</strong><br />
	August 2, 2013<br />
	<strong>Data Contest Deadline</strong><br />
	August 2, 2013<br />
	<strong>Redesign Contest Deadline</strong><br />
	August 2, 2013</p>
  </div>
</div>
  </div>
      </div></div> <!-- /.section, /#sidebar-second -->
    
  </div></div> <!-- /#main, /#main-wrapper -->

  
  <div id="footer-wrapper"><div class="section">

    
          <div id="footer" class="clearfix">
          <div class="region region-footer">
    <div id="block-block-13" class="block block-block">

    
  <div class="content">
    <p>BioVis 2013 is sponsored by the <a class="link" href="http://www.vgtc.org/">IEEE VGTC</a>. To contact the organizers please send an email to <a class="link" href="mailto:contact@biovis.net">contact@biovis.net</a>. &copy; 2013. BioVis. All Rights Reserved.</p>
  </div>
</div>
  </div>
      </div> <!-- /#footer -->
    
  </div></div> <!-- /.section, /#footer-wrapper -->

</div></div> <!-- /#page, /#page-wrapper -->
  </body>
</html>
