<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN"
  "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" version="XHTML+RDFa 1.0" dir="ltr"
  xmlns:content="http://purl.org/rss/1.0/modules/content/"
  xmlns:dc="http://purl.org/dc/terms/"
  xmlns:foaf="http://xmlns.com/foaf/0.1/"
  xmlns:og="http://ogp.me/ns#"
  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
  xmlns:sioc="http://rdfs.org/sioc/ns#"
  xmlns:sioct="http://rdfs.org/sioc/types#"
  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
  xmlns:xsd="http://www.w3.org/2001/XMLSchema#">

<head profile="http://www.w3.org/1999/xhtml/vocab">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta about="/year/2014/info/program_invited_speakers" property="sioc:num_replies" content="0" datatype="xsd:integer" />
<link rel="shortcut icon" href="../sites/all/themes/biovis/favicon.ico" type="image/vnd.microsoft.icon" />
<meta content="Invited Speakers" about="/year/2014/info/program_invited_speakers" property="dc:title" />
<link rel="shortlink" href="593.html" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="../year/2014/info/program_invited_speakers.html" />
  <title>Invited Speakers | BioVis</title>
  <style type="text/css" media="all">@import url("http://www.biovis.net/modules/system/system.base.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.menus.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.messages.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.theme.css?njdarp");</style>
<style type="text/css" media="all">@import url("http://www.biovis.net/modules/comment/comment.css?njdarp");
@import url("../sites/all/modules/date/date_api/date.css@njdarp.css");
@import url("../sites/all/modules/date/date_popup/themes/datepicker.1.7.css@njdarp.css");
@import url("http://www.biovis.net/modules/field/theme/field.css?njdarp");
@import url("http://www.biovis.net/modules/node/node.css?njdarp");
@import url("http://www.biovis.net/modules/search/search.css?njdarp");
@import url("http://www.biovis.net/modules/user/user.css?njdarp");
@import url("../sites/all/modules/views/css/views.css@njdarp.css");</style>
<style type="text/css" media="all">@import url("../sites/all/modules/ckeditor/ckeditor.css@njdarp.css");
@import url("../sites/all/modules/ctools/css/ctools.css@njdarp.css");</style>
<style type="text/css" media="all">@import url("http://www.biovis.net/themes/bartik/css/layout.css?njdarp");
@import url("http://www.biovis.net/themes/bartik/css/style.css?njdarp");
@import url("../sites/default/files/color/biovis-8495a6ec/colors.css@njdarp.css");
@import url("../sites/all/themes/biovis/css/biovis.css@njdarp.css");</style>
<style type="text/css" media="print">@import url("http://www.biovis.net/themes/bartik/css/print.css?njdarp");</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="http://www.biovis.net/sites/all/themes/biovis/css/ie.css?njdarp" media="all" />
<![endif]-->

<!--[if IE 6]>
<link type="text/css" rel="stylesheet" href="http://www.biovis.net/sites/all/themes/biovis/css/ie6.css?njdarp" media="all" />
<![endif]-->
  <script type="text/javascript" src="http://www.biovis.net/misc/jquery.js?v=1.4.4"></script>
<script type="text/javascript" src="http://www.biovis.net/misc/jquery.once.js?v=1.2"></script>
<script type="text/javascript" src="http://www.biovis.net/misc/drupal.js?njdarp"></script>
<script type="text/javascript" src="../sites/all/modules/google_analytics/googleanalytics.js@njdarp"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _gaq = _gaq || [];_gaq.push(["_setAccount", "UA-22015250-1"]);_gaq.push(["_trackPageview"]);(function() {var ga = document.createElement("script");ga.type = "text/javascript";ga.async = true;ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";var s = document.getElementsByTagName("script")[0];s.parentNode.insertBefore(ga, s);})();
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"biovis","theme_token":"EvoNbm2U8e5fGbhuWJT8jz2YkXVilA9OQE8w77Exs1w","js":{"misc\/jquery.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/google_analytics\/googleanalytics.js":1,"0":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/date\/date_api\/date.css":1,"sites\/all\/modules\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/views\/css\/views.css":1,"sites\/all\/modules\/ckeditor\/ckeditor.css":1,"sites\/all\/modules\/ctools\/css\/ctools.css":1,"themes\/bartik\/css\/layout.css":1,"themes\/bartik\/css\/style.css":1,"sites\/all\/themes\/biovis\/css\/colors.css":1,"sites\/all\/themes\/biovis\/css\/biovis.css":1,"themes\/bartik\/css\/print.css":1,"sites\/all\/themes\/biovis\/css\/ie.css":1,"sites\/all\/themes\/biovis\/css\/ie6.css":1}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls|xml|z|zip"}});
//--><!]]>
</script>
  <base href="" />
</head>
<body class="html not-front not-logged-in one-sidebar sidebar-second page-node page-node- page-node-593 node-type-conference-info" >
  <div id="skip-link">
    <a href="../index.html#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
    <div id="page-wrapper"><div id="page">

  <div id="header" class="without-secondary-menu"><div class="section clearfix">

          <a href="../year/2015/about.html" title="Home" rel="home" id="logo">
        <img src="../sites/default/files/logo_3.png" alt="Home" />
      </a>
    
          <div id="name-and-slogan">

                              <div id="site-name">
              <strong>
                <span class="conference-name">5th Symposium on Biological Data Visualization</span>
              </strong>
			  <br />
			    <span class="conference-info">10- 11 July // Dublin, Ireland @ ISMB/ECCB 2015</span>
            </div>
                  
        
      </div> <!-- /#name-and-slogan -->
				  <div id="header-rotation">
		    <img src="../sites/all/themes/biovis/header_rotation/2015/BioVisPoster0111-file1.png" alt=""/>		  </div>
		    
    
          <div id="main-menu" class="navigation">
	  <h2 class="element-invisible">Main menu</h2><ul id="main-menu-links" class="links clearfix"><li class="menu-592 first"><a href="../year/2015/about.html" title="">About</a></li>
<li class="menu-610"><a href="../year/2015/news.html" title="News">News</a></li>
<li class="menu-596"><a href="../year/2015/cfp.html" title="">CFP</a></li>
<li class="menu-597"><a href="../year/2015/program.html" title="">Program</a></li>
<li class="menu-598"><a href="../year/2015/papers.html" title="">Papers</a></li>
<li class="menu-599"><a href="../year/2015/posters/index.html" title="">Posters</a></li>
<li class="menu-601"><a href="../year/2015/design/update.html" title="">Design Contest</a></li>
<li class="menu-600"><a href="../year/2015/info/overview.html" title="">Data Contest</a></li>
<li class="menu-603"><a href="../year/2015/registration.html" title="Registration page">Registration</a></li>
<li class="menu-594"><a href="../year/2015/info/2015-committee.html" title="">Committee</a></li>
<li class="menu-595 last"><a href="../conference-archive.html" title="">Archive</a></li>
</ul>      </div> <!-- /#main-menu -->
    

    
  </div></div> <!-- /.section, /#header -->
          <div id="sub-menu">
          <div class="region region-sub-menu">
    <div id="block-menu-block-3" class="block block-menu-block">

    <h2>Program</h2>
  
  <div class="content">
    <div class="menu-block-wrapper menu-block-3 menu-name-menu-main-menu-2014 parent-mlid-0 menu-level-2">
  <ul class="menu clearfix"><li class="first leaf active-trail active menu-mlid-585"><a href="../year/2014/info/program_invited_speakers.html" title="" class="active-trail active">Invited Speakers</a></li>
<li class="last leaf menu-mlid-586"><a href="../year/2014/info/program_schedule.html" title="">Schedule</a></li>
</ul></div>
  </div>
</div>
  </div>
      </div> <!-- /.section, /#sub-menu -->
    
  
  
  <div id="main-wrapper" class="clearfix"><div id="main" class="clearfix">

    
    
    <div id="content" class="column"><div class="section">
            <a id="main-content"></a>
                    <h1 class="title" id="page-title">
          Invited Speakers        </h1>
                          <div class="tabs">
                  </div>
                          <div class="region region-content">
    <div id="block-system-main" class="block block-system">

    
  <div class="content">
    <div id="node-593" class="node node-conference-info node-full clearfix" about="/year/2014/info/program_invited_speakers" typeof="sioc:Item foaf:Document">

      
  
  <div class="content clearfix">
    <div class="field field-name-body field-type-text-with-summary field-label-hidden"><div class="field-items"><div class="field-item even" property="content:encoded"><h2>
	<span style="font-size: 13px; font-weight: normal;">We have selected a range of speakers that, together with the keynote speaker, will cover developments in a broad range of active research topics in modern biological data visualization.</span></h2>
<h2>
	Keynote</h2>
<h3>
	<a name="keynote">Visualization Analysis and Design for Biology</a><a href="http://www.cs.ubc.ca/~tmm/talks.html#biovis14">[slides]</a></h3>
<p>Computer-based visualization (vis) systems provide visual representations of datasets designed to help people carry out tasks more effectively. Visualization is suitable when there is a need to augment human capabilities rather than replace people with computational decision- making methods. The design space of possible vis idioms is huge, and includes the considerations of both how to create and how to interact with visual representations. Vis design is full of trade-offs, and most possibilities in the design space are ineffective for a particular task, so validating the effectiveness of a design is both necessary and difficult. Vis designers must take into account three very different kinds of resource limitations: those of computers, of humans, and of displays. Vis usage can be analyzed in terms of why the user needs it, what data is shown, and how the idiom is designed. I will discuss this framework for analyzing the design of visualization systems and analyze a range of biology visualization applications through this lens.</p>
<div>
	&nbsp;</div>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="" src="http://www.cs.ubc.ca/~tmm/tmmpix/TamaraMunzner-UBCSynergy2010Nov.med.png" style="width: 150px; height: 120px;" />
				<div>
					&nbsp;</div>
			</td>
			<td>
				<p><a href="http://www.nature.com/nmeth/about/about_eds/index.html#evanko">T</a><a href="http://www.cs.ubc.ca/~tmm/">amara Munzner</a><br />
					Professor at the Department of Computer Science, University of British Columbia</p>
				<hr />
				<p>Tamara Munzner is a Professor of Computer Science at the University of British Columbia, in the research area of information visualization. She has been active in visualization research since 1991, and has published over fifty papers and book chapters. Before earning a PhD from Stanford in 2000, she was a technical staff member at the NSF-funded Geometry Center at the University of Minnesota for four years. Afterwards, she was a research scientist at the Compaq Systems Research Lab from 2000 to 2002. She co-chaired InfoVis in 2003 and 2004 and EuroVis in 2009 and 2010. She is currently on the InfoVis Steering Committee and the VIS Executive Committee, was a founding member of the BioVis Steering Committee from 2010 through 2013, and was a Member At Large of the Executive Committee of the IEEE Visualization and Graphics Technical Committee from 2004-2009. She hasconsulted for companies including Silicon Graphics, Microsoft, and early-stage startups. Her research interests include the development, evaluation, and characterization of information visualization systems and techniques from both problem-driven and technique-driven perspectives. She has worked on problem-driven visualization projects in a broad range of application domains, including genomics, evolutionary biology, large-scale system administration, web loganalysis, computational linguistics, and geometric topology. Her technique-driven interests include dimensionality reduction and graph drawing.</p>
			</td>
		</tr>
	</tbody>
</table>
<h2>
	Primer Speakers</h2>
<h3>
	<a name="primer1">Presenting data</a></h3>
<p>Data visualization can serve two distinct purposes: to communicate research findings and to guide the data-exploration process as the scientific story is unfolding. Each goal requires a different approach to data representation, but sound graphic design principles are important in both. In this presentation I will demonstrate core concepts that make design a requirement, not a cosmetic addition, to construct presentations that are easy to interpret and understand.</p>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Bang Wong" src="http://vizbi.org/blog/wp-content/uploads/bang_wong.jpg" style="width: 150px; height: 217px;" /></td>
			<td>
				<p>Bang Wong<br />
					Creative Director,&nbsp;<a href="http://www.broadinstitute.org">Broad Institute of MIT</a></p>
				<hr />
				<p>Bang Wong is the creative director of the Broad Institute of MIT and Harvard and an adjunct assistant professor in the Department of Art as Applied to Medicine at the Johns Hopkins University School of Medicine. His work focuses on the effective communication, presentation and exploration of complex biomedical data. At the Broad he co-leads the Visualization Skunkworks, an initiative that brings together researchers to develop and explore visualization solutions to enable new fundamental research discoveries. Before starting the Skunkworks, he created an exhibition space call the DNAtrium which enables visitors to explore the human genome. He is the founding author of the popular Points of View column that deals with the fundamental aspects of data presentation published by Nature Methods. His work has appeared in the New York Times Nature, Cell, and Science. Bang received a Masters degree in Immunology and a Masters degree in Medical Illustration both from the Johns Hopkins University School of Medicine. He is an elected member of the Board of Governors for the Association of Medical Illustrators and the co-founder of the Visualizing Biological Data (VIZBI) conference series.&nbsp;</p>
			</td>
		</tr>
	</tbody>
</table>
<h3>
	<a name="primer2">Fundamentals of Visualizing Biological Data</a><a href="http://caleydo.icg.tugraz.at/publication/2014-07-11_BioVis_Primer.pdf">[slides]</a></h3>
<p>With the introduction of modern data acquisition techniques, like next-generation sequencing, life science researchers are confronted with an exploding collection of heterogeneous data. The availability of such techniques at comparably low costs shifted the challenge from the acquisition of the data to its analysis. However, not only the sheer amount of the data but also its complexity is challenging for life science researchers. While computational techniques have a long tradition in analyzing biological data, interactive visualization is now recognized as an important building block of the analysis process. Visualization can make use of the unique ability of humans to see patterns and trends in the data. In this presentation I will first give a broad overview of state-of-the-art techniques for visualizing biological data. In the second part of the talk I will focus on approaches that the visualization community developed to cope with the particular challenges posed by large and heterogeneous data.</p>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Marc Streit" src="../sites/default/files/pictures/marc_streit.jpg" style="width: 150px; height: 217px;" /></td>
			<td>
				<p>Marc Streit<br />
					Assistant Professor,&nbsp;<a href="http://www.jku.at/cg/content/e48314/index_html?team_view=section&amp;emp=e48314/employee_groups_wiss152471/employees140191">Institute of Computer Graphics</a>, Johannes Kepler University Linz, Austria</p>
				<hr />
				<p>Marc Streit is assistant professor at the Institute of Computer Graphics, Johannes Kepler University Linz, Austria. He finished his PhD at Graz University of Technology in early 2011 and moved to Linz later that year. As part of his tenure-track position, he spent a part of the year 2012 as a visiting researcher at the Center for Biomedical Informatics at Harvard Medical School. In 2014 he received a Fulbright scholarship for research and lecturing at Harvard University. He has won Best Paper Awards at InfoVis&#39;13, BioVis&#39;12, InfoVis&#39;11, GI&#39;10 and Honorable Mention Awards at CHI&#39;14 and EuroVis&#39;12. He is a co-author of the Nature Methods &quot;Points of View&quot; column. In 2013 he co-edited the Special Issue on Visual Analytics in the IEEE Computer journal. Additionally, he is actively contributing to the scientific community by serving on the organizing and program committee of several scientific events as well as by acting as a reviewer for high-quality journals and conferences. His scientific areas of interest include Information Visualization, Visual Analytics, and Biological Data Visualization, where he is particularly interested in the integrated analysis of large heterogeneous data.</p>
			</td>
		</tr>
	</tbody>
</table>
<h2>
	Challenges Speakers</h2>
<h3>
	<a name="challenge1">Interactive graphics for high-dimensional genetic data</a><a href="https://www.biostat.wisc.edu/~kbroman/presentations/BioVis/">[slides]</a></h3>
<div>
	The value of interactive, dynamic graphics for making sense of high-dimensional data has long been appreciated but is still not in routine use. &nbsp;I will describe my efforts to develop interactive graphical tools for genetic data, using JavaScript and D3. &nbsp;I will focus on an expression genetics experiment in the mouse, with gene expression microarray data on each of six tissues, plus high-density genotype data, in each of 500 mice. &nbsp;I argue that in research with such data, precise statistical inference is not so important as data visualization.</div>
<div>
	&nbsp;</div>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="" src="http://www.biostat.wisc.edu/~kbroman/pictures/karl_2014-03-30_crop.jpg" style="height: 176px; width: 150px;" /></td>
			<td>
				<p><a href="http://www.biostat.wisc.edu/~kbroman/">Karl W. Broman</a><br />
					Professor, Department of Biostatistics &amp; Medical Informatics, School of Medicine and Public Health, University of Wisconsin-Madison</p>
				<hr />
				<div>
					Karl Broman is Professor in the Department of Biostatistics &amp; Medical Informatics, University of Wisconsin-Madison. &nbsp;He is an applied statistician working on problems in genetics and genomics, with a particular focus on the genetic dissection of complex traits in experimental organisms. He develops the R package, R/qtl (<a href="http://rqtl.org">rqtl.org</a>).</div>
			</td>
		</tr>
	</tbody>
</table>
<h3>
	<a name="challenge2">Visualization Challenges and Advances in Metagenomics</a></h3>
<div>
	Over the last fifteen years culture independent methods enabled sampling and analyzing microbial communities directly from the environment. Metagenomics field quickly thrived from the initial studies of simple communities to global oceans and human microbiome with advances in sequencing technologies and computational tools. Despite number of advances in the field, it still poses a significant challenge to effectively visualize metagenomics data. Most visualization tools are developed for genomics studies and they fall short in representing high volume, high dimensional metagenomics studies. Especially the magnitude of the data presents a challenge to meaningfully represent biologically valuable information from complex analysis results. In this talk, I will discuss the progress and challenges of metagenomics data visualization and highlight emerging tools. I will also give my perspective on using simple model communities to develop visualization tools for metagenomics.</div>
<div>
	&nbsp;</div>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="" src="http://www.systemsbiology.org/sites/default/files/staff_images/STurkarslan.jpg" style="height: 176px; width: 150px;" /></td>
			<td>
				<p><a href="http://www.systemsbiology.org/serdar-turkarslan">Serdar Turkarslan</a><br />
					Research Scientist, Institute for Systems Biology, Seattle, Washington</p>
				<hr />
				<div>
					Dr. Turkarslan is a Research Scientist at Institute for Systems Biology in Seattle, Washington. He got his PhD in Cell and Molecular Biology at University of Pennsylvania. He is interested in understanding how cells respond to changes in the environment by reprogramming their gene regulatory networks (GRN) and delineating the principles underlying the evolution of these complex regulatory programs. Dr. Turkarslan uses systems approaches to build global scale predictive network models for microbial organisms and simple communities to reverse engineer biological circuits. Currently, he is focused on delineating regulatory network readjustments that drive evolution of syntrophic community between sulfate reducing bacteria and methanogenic archaea. He is also interested in developing tools for exploration and visualization of large datasets in the field of systems biology and metagenomics. Dr. Turkarslan recently led development of the Network Portal Framework for storing, analyzing and visualizing biological networks.</div>
				<div>
					&nbsp;</div>
			</td>
		</tr>
	</tbody>
</table>
<p>&nbsp;</p>
</div></div></div>  </div>

  
  
</div>
  </div>
</div>
  </div>
      
    </div></div> <!-- /.section, /#content -->

          <div id="sidebar-second" class="column sidebar"><div class="section">
          <div class="region region-sidebar-second">
    <div id="block-block-23" class="block block-block">

    
  <div class="content">
    <p>BioVis 2014 is an official SIG of ISMB</p>
<p><a href="http://www.iscb.org"><img alt="" src="../sites/default/files/iscb_logo_sm.png" style="width: 146px; height: 50px;" /></a></p>
<p><strong>BioVis 2014 Supporters</strong></p>
<p>Platinum:</p>
<p><img alt="Agilent" src="../sites/default/files/4c_CorporateSig-noHL_0.jpg" style="width: 146px; height: 28px;" /></p>
<p>Silver:</p>
<p><img alt="" src="../sites/default/files/autodesk_logo_sm.png" style="width: 146px; height: 30px;" /></p>
<p><a href="http://www.mathmed.org/"><img alt="Agilent" src="../sites/default/files/bcmm_logo_sm.png" style="width: 146px; height: 54px;" /></a></p>
<div>
	&nbsp;</div>
<div>
	We also acknowledge the support by <strong>BMC.</strong></div>
<div>
	&nbsp;</div>
<div>
	<a href="http://www.almob.org/" target="_blank"><img alt="" src="http://www.almob.org/sites/10104/images/logo.gif" style="width: 141px; height: 33px;" /></a></div>
  </div>
</div>
<div id="block-block-19" class="block block-block">

    <h2>Follow BioVis on Twitter</h2>
  
  <div class="content">
    <p><a href="http://twitter.com/#!/search/realtime/%23biovis">@biovis_net / #biovis</a></p>
  </div>
</div>
  </div>
      </div></div> <!-- /.section, /#sidebar-second -->
    
  </div></div> <!-- /#main, /#main-wrapper -->

  
  <div id="footer-wrapper"><div class="section">

    
          <div id="footer" class="clearfix">
          <div class="region region-footer">
    <div id="block-block-20" class="block block-block">

    
  <div class="content">
    <p>BioVis 2014.&nbsp;To contact the organizers please send an email to&nbsp;<a class="link" href="mailto:contact@biovis.net">contact@biovis.net</a>. &copy; 2014. BioVis. All Rights Reserved.</p>
  </div>
</div>
  </div>
      </div> <!-- /#footer -->
    
  </div></div> <!-- /.section, /#footer-wrapper -->

</div></div> <!-- /#page, /#page-wrapper -->
  </body>
</html>
