<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN"
  "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" version="XHTML+RDFa 1.0" dir="ltr"
  xmlns:content="http://purl.org/rss/1.0/modules/content/"
  xmlns:dc="http://purl.org/dc/terms/"
  xmlns:foaf="http://xmlns.com/foaf/0.1/"
  xmlns:og="http://ogp.me/ns#"
  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
  xmlns:sioc="http://rdfs.org/sioc/ns#"
  xmlns:sioct="http://rdfs.org/sioc/types#"
  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
  xmlns:xsd="http://www.w3.org/2001/XMLSchema#">

<head profile="http://www.w3.org/1999/xhtml/vocab">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta about="/year/2015/program" property="sioc:num_replies" content="0" datatype="xsd:integer" />
<link rel="shortcut icon" href="../../sites/all/themes/biovis/favicon.ico" type="image/vnd.microsoft.icon" />
<meta content="Program" about="/year/2015/program" property="dc:title" />
<link rel="shortlink" href="../../node/671.html" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="program.html" />
  <title>Program | BioVis</title>
  <style type="text/css" media="all">@import url("http://www.biovis.net/modules/system/system.base.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.menus.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.messages.css?njdarp");
@import url("http://www.biovis.net/modules/system/system.theme.css?njdarp");</style>
<style type="text/css" media="all">@import url("http://www.biovis.net/modules/comment/comment.css?njdarp");
@import url("../../sites/all/modules/date/date_api/date.css@njdarp.css");
@import url("../../sites/all/modules/date/date_popup/themes/datepicker.1.7.css@njdarp.css");
@import url("http://www.biovis.net/modules/field/theme/field.css?njdarp");
@import url("http://www.biovis.net/modules/node/node.css?njdarp");
@import url("http://www.biovis.net/modules/search/search.css?njdarp");
@import url("http://www.biovis.net/modules/user/user.css?njdarp");
@import url("../../sites/all/modules/views/css/views.css@njdarp.css");</style>
<style type="text/css" media="all">@import url("../../sites/all/modules/ckeditor/ckeditor.css@njdarp.css");
@import url("../../sites/all/modules/ctools/css/ctools.css@njdarp.css");</style>
<style type="text/css" media="all">@import url("http://www.biovis.net/themes/bartik/css/layout.css?njdarp");
@import url("http://www.biovis.net/themes/bartik/css/style.css?njdarp");
@import url("../../sites/default/files/color/biovis-8495a6ec/colors.css@njdarp.css");
@import url("../../sites/all/themes/biovis/css/biovis.css@njdarp.css");</style>
<style type="text/css" media="print">@import url("http://www.biovis.net/themes/bartik/css/print.css?njdarp");</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="http://www.biovis.net/sites/all/themes/biovis/css/ie.css?njdarp" media="all" />
<![endif]-->

<!--[if IE 6]>
<link type="text/css" rel="stylesheet" href="http://www.biovis.net/sites/all/themes/biovis/css/ie6.css?njdarp" media="all" />
<![endif]-->
  <script type="text/javascript" src="http://www.biovis.net/misc/jquery.js?v=1.4.4"></script>
<script type="text/javascript" src="http://www.biovis.net/misc/jquery.once.js?v=1.2"></script>
<script type="text/javascript" src="http://www.biovis.net/misc/drupal.js?njdarp"></script>
<script type="text/javascript" src="../../sites/all/modules/google_analytics/googleanalytics.js@njdarp"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _gaq = _gaq || [];_gaq.push(["_setAccount", "UA-22015250-1"]);_gaq.push(["_trackPageview"]);(function() {var ga = document.createElement("script");ga.type = "text/javascript";ga.async = true;ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";var s = document.getElementsByTagName("script")[0];s.parentNode.insertBefore(ga, s);})();
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"biovis","theme_token":"0YxCgN6fmTyeeQy64_iBOvBYa-6NnnRtPf1GdeL39xM","js":{"misc\/jquery.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/google_analytics\/googleanalytics.js":1,"0":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/date\/date_api\/date.css":1,"sites\/all\/modules\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/views\/css\/views.css":1,"sites\/all\/modules\/ckeditor\/ckeditor.css":1,"sites\/all\/modules\/ctools\/css\/ctools.css":1,"themes\/bartik\/css\/layout.css":1,"themes\/bartik\/css\/style.css":1,"sites\/all\/themes\/biovis\/css\/colors.css":1,"sites\/all\/themes\/biovis\/css\/biovis.css":1,"themes\/bartik\/css\/print.css":1,"sites\/all\/themes\/biovis\/css\/ie.css":1,"sites\/all\/themes\/biovis\/css\/ie6.css":1}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls|xml|z|zip"}});
//--><!]]>
</script>
  <base href="" />
</head>
<body class="html not-front not-logged-in one-sidebar sidebar-second page-node page-node- page-node-671 node-type-page" >
  <div id="skip-link">
    <a href="../../index.html#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
    <div id="page-wrapper"><div id="page">

  <div id="header" class="without-secondary-menu"><div class="section clearfix">

          <a href="about.html" title="Home" rel="home" id="logo">
        <img src="../../sites/default/files/logo_3.png" alt="Home" />
      </a>
    
          <div id="name-and-slogan">

                              <div id="site-name">
              <strong>
                <span class="conference-name">5th Symposium on Biological Data Visualization</span>
              </strong>
			  <br />
			    <span class="conference-info">10- 11 July // Dublin, Ireland @ ISMB/ECCB 2015</span>
            </div>
                  
        
      </div> <!-- /#name-and-slogan -->
				  <div id="header-rotation">
		    <img src="../../sites/all/themes/biovis/header_rotation/2015/BioVisPoster0119-file1.png" alt=""/>		  </div>
		    
    
          <div id="main-menu" class="navigation">
	  <h2 class="element-invisible">Main menu</h2><ul id="main-menu-links" class="links clearfix"><li class="menu-592 first"><a href="about.html" title="">About</a></li>
<li class="menu-610"><a href="news.html" title="News">News</a></li>
<li class="menu-596"><a href="cfp.html" title="">CFP</a></li>
<li class="menu-597 active-trail active"><a href="program.html" title="" class="active-trail active">Program</a></li>
<li class="menu-598"><a href="papers.html" title="">Papers</a></li>
<li class="menu-599"><a href="posters/index.html" title="">Posters</a></li>
<li class="menu-601"><a href="design/update.html" title="">Design Contest</a></li>
<li class="menu-600"><a href="info/overview.html" title="">Data Contest</a></li>
<li class="menu-603"><a href="registration.html" title="Registration page">Registration</a></li>
<li class="menu-594"><a href="info/2015-committee.html" title="">Committee</a></li>
<li class="menu-595 last"><a href="../../conference-archive.html" title="">Archive</a></li>
</ul>      </div> <!-- /#main-menu -->
    

    
  </div></div> <!-- /.section, /#header -->
    
  
  
  <div id="main-wrapper" class="clearfix"><div id="main" class="clearfix">

    
    
    <div id="content" class="column"><div class="section">
            <a id="main-content"></a>
                    <h1 class="title" id="page-title">
          Program        </h1>
                          <div class="tabs">
                  </div>
                          <div class="region region-content">
    <div id="block-system-main" class="block block-system">

    
  <div class="content">
    <div id="node-671" class="node node-page node-full clearfix" about="/year/2015/program" typeof="foaf:Document">

      
  
  <div class="content clearfix">
    <div class="field field-name-body field-type-text-with-summary field-label-hidden"><div class="field-items"><div class="field-item even" property="content:encoded"><p><a href="../../sites/default/files/ProgramBioVis15_4print.pdf">BioVis&#39;15 Program and Schedule (short version pdf)&nbsp;</a></p>
<p><a href="http://biovis.net/sites/default/files/ISMBProgramBioVis2015_0.pdf">BioVis&#39;15 Program and Schedule book (long version pdf)</a></p>
<p>The final papers will be available under the following links:</p>
<p><a href="http://www.biomedcentral.com/bmcbioinformatics/supplements/16/S11">BMC Bioinformatics</a></p>
<p><a href="http://www.biomedcentral.com/bmcproc/supplements/9/S6">BMC Proceedings</a></p>
<p>&nbsp;</p>
<h1>
	<strong>Keynote Speakers</strong></h1>
<p>BioVis&#39;15 features one keynote talks: a visualization keynote talk by <strong><em>Torsten Moller</em></strong> (University of Vienna) on Friday, July 10th.</p>
<p>Unfortunately the announced keynote by Chris Sander had to be cancelled.</p>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<p><img alt="Min Chen" height="280" src="../../sites/default/files/pictures/TorstenMoller.jpg" width="191" /></p>
				<p>Copyright &copy; Barbara Mair</p>
			</td>
			<td>
				<h2>
					<span style="font-family:'Helvetica'">Torsten M&ouml;ller</span></h2>
				<p>Head of research group of Visualization and Data Analysis<br />
					University of Vienna, Austria</p>
				<hr />
				<p><strong>Visual data science -- Advancing science through visual reasoning</strong></p>
				<p>Modern science is driven by computers (computational science) and data (data-driven science). While visual analysis has always been an integral part of science, in the context of computational science and data-driven science it has gained new importance. In this talk I will demonstrate novel approaches in visualization to support the process of modeling and simulations. Especially, I will report on some of the latest approaches and challenges in modeling and reasoning with uncertainty. Visual tools for ensemble analysis, sensitivity analysis, and the cognitive challenges during decision making build the basis of an emerging field of visual data science which is becoming an essential ingredient of computational thinking.</p>
				<hr />
				<p><strong>Biography</strong></p>
				<p>Torsten M&ouml;ller is a professor at the University of Vienna, Austria, since 2013. Between 1999 and 2012 he served as a Computing Science faculty member at Simon Fraser University, Canada. He received his PhD in Computer and Information Science from Ohio State University in 1999 and a Vordiplom (BSc) in mathematical computer science from Humboldt University of Berlin, Germany. He is a senior member of IEEE and ACM, and a member of Eurographics. His research interests include algorithms and tools for analyzing and displaying data with principles rooted in computer graphics, image processing, visualization and human-computer interaction.<br />
					<br />
					He heads the research group of Visualization and Data Analysis. He served as the appointed Vice Chair for Publications of the IEEE Visualization and Graphics Technical Committee (VGTC) between 2003 and 2012. He has served on a number of program committees and has been papers co-chair for IEEE Visualization, EuroVis, Graphics Interface, and the Workshop on Volume Graphics as well as the Visualization track of the 2007 International Symposium on Visual Computing. He has also co-organized the 2004 Workshop on Mathematical Foundations of Scientific Visualization, Computer Graphics, and Massive Data Exploration as well as the 2010 Workshop on Sampling and Reconstruction: Applications and Advances at the Banff International Research Station, Canada. He is a co-founding chair of the Symposium on Biological Data Visualization (BioVis). In 2010, he was the recipient of the NSERC DAS award. He received best paper awards from IEEE Conference on Visualization (1997), Symposium on Geometry Processing (2008), and EuroVis (2010), as well as two second best paper awards from EuroVis (2009, 2012).</p>
			</td>
		</tr>
	</tbody>
</table>
<p>&nbsp;</p>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<p>&nbsp;</p>
				<p><img alt="" src="../../sites/default/files/PastedGraphic-2.png" style="width: 191px; height: 127px;" /></p>
				<p>&nbsp;</p>
			</td>
			<td>
				<h2>
					<span style="font-family: Helvetica;">Larry Hunter</span></h2>
				<p>Director of University of Colorado&#39;s Computational Bioscience Program</p>
				<hr />
				<p><strong>The challenges of biological information visualization</strong></p>
				<p>Interpretation of contemporary molecular assays (e.g. genome-wide studies of polymorphisms or gene expression) is a daunting challenge. Typical genome-scale experiments implicate dozens or hundreds of genes, gene products, or polymorphisms. The volume of information relevant to those genes can be overwhelming; the average human gene now has more than 2,000 publications about it. In order to benefit fully from existing knowledge, researchers must assimilate extensive information from many domains, often well outside their own areas of expertise. The amount, diversity, and complexity of all this potentially relevant information puts full exploitation of it beyond the time and attention capacity of even dedicated researchers. Failing to capitalize on all this relevant existing knowledge substantially reduces the efficiency of the entire biomedical research enterprise, delaying the development of both key insights and new therapies. The information visualization community needs to develop new user-specific information visualization approaches that focus biomedical researchers&rsquo; attention on the material most interesting to them. This talk will describe both these challenges and some possible approaches to dealing with them.</p>
				<hr />
				<p><strong>Biography</strong></p>
				<p>Dr. Lawrence Hunter is the Director of the University of Colorado&#39;s Computational Bioscience Program and a Professor of Pharmacology (School of Medicine) and Computer Science (Boulder). He received a Ph.D. in computer science from Yale University in 1989, and then joined the National Institutes of Health as a staff scientist, first at the National Library of Medicine and then at the National Cancer Institute, before coming to Colorado in 2000. Dr. Hunter is widely recognized as one of the founders of bioinformatics; he served as the first President of the International Society for Computational Biology (ISCB), and created several of the most important conferences in the field, including ISMB, PSB and VizBi . Dr. Hunter&#39;s research interests span a wide range of areas, from cognitive science to rational drug design. He has published more than 100 scientific papers, holds two patents and has been elected a fellow of both the ISCB and the American College of Medical Informatics. His primary focus recently has been the integration of natural language processing, knowledge representation, machine learning and advanced visualization techniques to address challenges in interpreting data generated by high throughput molecular biology.</p>
			</td>
		</tr>
	</tbody>
</table>
<h1>
	&nbsp;</h1>
<p>* <span style="color:#800000;">Chris Sander&#39;s talk is cancelled&nbsp;due to unforeseen personal circumstances</span></p>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<p>&nbsp;</p>
				<p><img alt="" src="../../sites/default/files/chris.jpg" style="width: 191px; height: 127px;" /></p>
				<p>&nbsp;</p>
			</td>
			<td>
				<h2>
					<span style="font-family: Helvetica;">Chris Sander</span></h2>
				<p>Computational Biosystems, MSKCC</p>
				<hr />
				<p><strong>Solving complicated problems in knowledge representation and visualization: cBioPortal, Pathway Commons, Precision Medicine</strong></p>
				<hr />
				<p><strong>Biography</strong></p>
				<p>Chris Sander is the chair of the computational biology program at the Memorial Sloan Kettering Cancer Center in New York. His research focuses on analyzing and simulating biological processes at different levels of organization. This includes work on the identification of oncogenically altered pathways from genomic and molecular profiling in cancer, algorithms for the analysis of cancer genomics data, design of combinatorial cancer therapy, drug target identification, knowledge representation of biological pathways, protein evolution, specificity in protein networks, and the function of small RNAs.</p>
			</td>
		</tr>
	</tbody>
</table>
<h1>
	&nbsp;</h1>
<h1>
	<strong>Primers Speakers</strong></h1>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Min Chen" height="192" src="../../sites/default/files/pictures/MinChen300x300.jpg" width="192" /></td>
			<td>
				<h2>
					<strong>Min Chen</strong></h2>
				<p>BSc, PhD, FBCS, FEG, FLSW<br />
					Fellow of Pembroke College<br />
					Oxford e-Research Centre<br />
					University of Oxford</p>
				<hr />
				<p><strong>Four Levels of Visualization</strong></p>
				<p>In this talk, the speaker will discuss four levels of visualization in the context of bioinformatics. The four levels are disseminative, observational, analytical and model-developmental visualization, which reflect different visualization tasks as well as different levels of complexity. The speaker will also briefly discuss the latest development of an information-theoretic measure for optimizing the cost-benefit ratio of a data analysis and visualization process.</p>
				<hr />
				<p><strong>Biography</strong></p>
				<p>Min Chen developed his academic career in Wales between 1984 and 2011. He is currently the professor of scientific visualization at Oxford University and a fellow of Pembroke College. His research interests include visualization, computer graphics and human-computer interaction. He has co-authored some 160 publications, including his recent contributions in areas of volume graphics, video visualization, face modelling, automated visualization and theory of visualization. He has been awarded over &pound;11M research grants from EPSRC, JISC (AHRC), TSB (NERC), Royal Academy, Welsh Assembly Government, HEFCW, Industry, and several UK and US Government Agencies. He is currently leading visualization activities at Oxford e-Research Centre, with a team of researchers working on a broad spectrum of interdisciplinary research topics, ranging from the sciences to sports, and from digital humanities to cybersecurity. His services to the research community include papers co-chair of IEEE Visualization 2007 and 2008, Eurographics 2011, IEEE VAST 2014 and 2015; co-chair of Volume Graphics 1999 and 2006, EuroVis 2014; associate editor-in-chief of IEEE Transactions on Visualization and Computer Graphics; and co-director of Wales Research Institute of Visual Computing. He is a fellow of British Computer Society, European Computer Graphics Association, and Learned Society of Wales.</p>
			</td>
		</tr>
	</tbody>
</table>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Robert Kosara" height="188" src="../../sites/default/files/pictures/Robert.jpg" width="192" /></td>
			<td>
				<h2>
					<strong>Robert Kosara</strong></h2>
				<p>Research Scientist<br />
					Tableau Software</p>
				<hr />
				<p><strong>How to Visualize Data</strong></p>
				<p>Visualization of data isn&rsquo;t just about creating pretty pictures for the cover of Nature, but really about being able to perceive and understand data. I will give a brief introduction to the basics of data visualization, show some common techniques and tools, and walk you through some examples. Presentation will not be ignored entirely, though it has to be preceded by exploration and analysis of the data.</p>
				<hr />
				<p><strong>Biography</strong></p>
				<p>Robert Kosara is a research scientist at Tableau Software. His focus is on the communication of data through&nbsp;visualization and visual storytelling. Robert is also working on furthering our understanding of visual perception&nbsp;and cognition, so we can make data easier to understand and develop tools to communicate it more effectively.&nbsp;His full list of publications can be found on&nbsp;<a href="http://kosara.net/publications.html" target="_blank">his vanity website</a>.</p>
				<p>Before joining Tableau in 2012, Robert was Associate Professor of Computer Science at The University of&nbsp;North Carolina at Charlotte. Robert received his M.Sc. and Ph.D. Degrees in Computer Science from Vienna&nbsp;University of Technology (Austria). In his copious spare time, Robert likes to run long distances and writing&nbsp;articles for his website,&nbsp;<a href="https://eagereyes.org" target="_blank">eagereyes.org</a>.</p>
			</td>
		</tr>
	</tbody>
</table>
<h1>
	<strong>Challenges Speakers</strong></h1>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Marc Marti-Renom" height="145" src="../../sites/default/files/pictures/nacho_ordenador_medium.jpeg" width="191" /></td>
			<td>
				<h2>
					<strong>Ignacio Medina</strong></h2>
				<p>Head of Computational Biology Lab<br />
					University of Cambridge</p>
				<hr />
				<p><strong>New Advanced Solutions for Genomic Big Data Analysis and Visualization</strong></p>
				<hr />
				<div>
					<strong>Biography</strong></div>
				<div>
					&nbsp;</div>
				<div>
					<span style="font-size: 12.8000001907349px;">Ignacio Medina received his BSc in Biochemistry and Molecular Biology from the University of&nbsp;</span>Valencia<span style="font-size: 12.8000001907349px;">, Spain, in 2001, and the BSc in Computer Science from the Polytechnic University of&nbsp;</span>Valencia<span style="font-size: 12.8000001907349px;">, in 2004. In 2009 he obtained the MSc in Genetics from University of Valencia. In 2006, he joined the Department of Bioinformatics and Genomics at the Prince Felipe Research Center, as a Bioinformatician and Researcher, and in 2010 became a Project Manager of several clinical and software development projects. In 2014, he joined EMBL-EBI as a Project Manager and Senior Software Architect at EBI Variation Team. In 2015, he joined&nbsp;</span><span style="font-size: 12.8000001907349px;">University of Cambridge as a</span><span style="font-size: 12.8000001907349px;">&nbsp;Head of Computational Biology Lab at HPC Service. His current research interests include HPC and big data software development for genomic-scale data analysis and visualization. He participates in several international projects such as EGA or Genomics England. He has authored more than 35 papers in international peer-reviewed journals.</span></div>
			</td>
		</tr>
	</tbody>
</table>
<table border="0" cellpadding="7" style="border: 0px solid gray;">
	<tbody>
		<tr align="" bgcolor="" dir="" id="" lang="" valign="">
			<td align="center" dir="" id="" lang="" scope="" valign="top">
				<img alt="Marc Marti-Renom" height="286" src="../../sites/default/files/pictures/MarcMartiRenom.jpg" width="191" /></td>
			<td>
				<h2>
					<strong>Marc Mart&iacute;-Renom</strong></h2>
				<p>ICREA Research Professor<br />
					Genome Biology Group, Centre Nacional d&#39;An&agrave;lis Gen&ograve;mica (CNAG,&nbsp;<a href="http://cnag.org/" style="color:purple" target="_blank">http://cnag.org</a>), Barcelona, Spain.<br />
					Structural Genomics Group, Centre de Regulaci&oacute; Gen&ograve;mica (CRG,&nbsp;<a href="http://crg.cat/" style="color:purple" target="_blank">http://crg.cat</a>), Barcelona, Spain.</p>
				<hr />
				<h3>
					<strong style="font-size: 13px;">Visualizing 3D Genomes </strong></h3>
				<p>The genome is conventionally represented and explored as a linear sequence, however chromatin is dynamic. The spatial organization and dynamics of the&nbsp; chromatin can be computationally modeled. The extent, detail and utility of these types of models present challenges in representing and interacting with such large, multi-scale models. Unfortunately, current genome browsers have limited overview 3D spatial data and do not provide an adequate end-user experience while browsing such generated models. New web-based, open-source, cross-platform technologies provide now the opportunity to address these issues with increasing ease, stability and security. Despite the limitations in three-dimensional (3D) representation due to occlusion, subjectivity and interpretation inherent in human system, 3D visualizations can assist navigability, comprehension and discovery by leveraging human visual processing, spatial reasoning and creativity. We will describe a new Web-based browser that aims at visualizing the genome from all its dimensions (linear to 3D). Our approach, called TADkit, aims at integrating &lsquo;1D&rsquo; genome sequence, its &lsquo;2D&rsquo; aligned annotations and its &lsquo;3D&rsquo; models to give a more complete vision of the forms and interactions.</p>
				<hr />
				<p><strong>Biography</strong></p>
				<p>Marc Marti-Renom obtained a Ph.D. in Biophysics from the UAB where he worked on protein folding under the supervision of Professors B. Oliva, F.X. Avil&eacute;s and M. Karplus. He conducted postdoctoral training on protein structure modeling at the Sali Lab (Rockefeller University) as the recipient of the Burroughs, Wellcome Fund fellowship and was then appointed Assistant Adjunct Professor at UCSF. Between 2006 and 2011, Prof. Marti-Renom headed the Structural Genomics Group at the CIPF in Valencia (Spain). Currently, he is a ICREA research professor and leads the Genome Biology Group at the National Center for Genomic Analysis (CNAG) and the Structural Genomics Group at the Centre for Genomic Regulation (CRG), both in Barcelona. His group is broadly interested on how RNA, proteins and genomes organize and regulate cell fate. He is an Associate Editor of the PLoS Computational Biology and BMC Structural Biology journals and has published over 75 articles in international peer-reviewed journals.</p>
			</td>
		</tr>
	</tbody>
</table>
<h3>
	&nbsp;</h3>
<p>&nbsp;</p>
</div></div></div>  </div>

  
  
</div>
  </div>
</div>
  </div>
      
    </div></div> <!-- /.section, /#content -->

          <div id="sidebar-second" class="column sidebar"><div class="section">
          <div class="region region-sidebar-second">
    <div id="block-block-26" class="block block-block">

    <h2>Sponsors</h2>
  
  <div class="content">
    <p>BioVis 2015 is an official SIG of <a href="http://www.iscb.org/ismbeccb2015">ISMB 2015</a></p>
<p><a href="http://www.iscb.org"><img alt="" height="70" src="../../sites/default/files/pictures/ISCB.jpg" width="163" /></a></p>
<p><strong>BioVis 2015 Supporters</strong></p>
<p>Platinum:</p>
<p><img alt="" height="26" src="../../sites/default/files/autodesk_header_logo_140x23.png" width="163" /></p>
<p>Bronze:</p>
<p><a href="http://www.mathmed.org/"><img alt="Agilent" height="63" src="../../sites/default/files/bcmm_logo_sm.png" width="162" /></a></p>
<p><a href="http://www.nvidia.com/content/global/global.php"><img alt="" height="80" src="../../sites/default/files/pictures/NVIDIA.png" width="110" /></a></p>
<p>We also acknowledge the support by&nbsp;<strong>BMC.&nbsp;</strong></p>
<p><a href="http://www.biomedcentral.com/bmcbioinformatics"><strong><img alt="" height="79" src="../../sites/default/files/pictures/publications_BMC.jpg" width="160" /></strong></a></p>
<p>Please check out our <a href="http://www.biomedcentral.com/bmcbioinformatics/series/BioVis2014">thematic series</a> in conjunction with BioVis 2014 and&nbsp;the <a href="http://www.biomedcentral.com/bmcproc/supplements/8/S2">proceedings</a> in conjunction with BioVis 2013</p>
<p><img alt="" height="45" src="../../sites/default/files/pictures/SpringerLogo.png" width="164" /></p>
  </div>
</div>
  </div>
      </div></div> <!-- /.section, /#sidebar-second -->
    
  </div></div> <!-- /#main, /#main-wrapper -->

  
  <div id="footer-wrapper"><div class="section">

    
          <div id="footer" class="clearfix">
          <div class="region region-footer">
    <div id="block-block-25" class="block block-block">

    
  <div class="content">
    <p>BioVis 2015.&nbsp;To contact the organizers please send an email to&nbsp;<a class="link" href="mailto:contact@biovis.net">contact@biovis.net</a>. &copy; 2015. BioVis. All Rights Reserved.</p>
  </div>
</div>
  </div>
      </div> <!-- /#footer -->
    
  </div></div> <!-- /.section, /#footer-wrapper -->

</div></div> <!-- /#page, /#page-wrapper -->
  </body>
</html>
