<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Papers BioVis@VIS</title>

  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css"/>
  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,300' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" href="/2016/css/main.css">
</head>


<body class="site">

    <div class="body">
    <nav>
        <div class="nav-wrapper">

            <ul id="vis" class="dropdown-content">
                <li><a href="/2016/ieeevis">Overview</a></li>
                <li class="divider">
                <li><a href="/2016/cfp_vis">Call for Participation</a></li>
                <li><a href="/2016/papers">Paper Submission</a></li>
                <li><a href="/2016/posters_ieee">Poster Submission</a></li>
                <li><a href="/2016/designContest_vis">Design Contest</a></li>
                <li class="divider"></li>
                <li><a href="/2016/program_ieee">Program</a></li>
            </ul>
            <ul id="ismb" class="dropdown-content">
                <li><a href="/2016/ismb">Overview</a></li>
                <li class="divider"></li>
                <li><a href="/2016/cfp_ismb">Call for Participation</a></li>
                <li><a href="/2016/submission_ismb">Abstract Submission</a></li>
                <li><a href="/2016/registration_ismb">Registration</a></li>
                <li class="divider"></li>
                <li><a href="/2016/program_ismb">Program</a></li>
                <!--<li><a href="/2016/posters_ismbAccepted">Accepted Posters</a></li>-->
            </ul>

            <a href="/2016/" class="brand-logo"><img src="/2016/images/logo-white.svg"
                                                                  width="150"/></a>
            <a href="#" data-activates="mobile" class="button-collapse"><i class="material-icons">menu</i></a>
            <ul class="right hide-on-med-and-down">
                <li><a class="dropdown-button" href="#!" data-beloworigin="true" data-activates="vis">@IEEE VIS<i
                        class="material-icons right">arrow_drop_down</i></a></li>
                <li><a class="dropdown-button" href="#!" data-beloworigin="true" data-activates="ismb">@ISMB<i
                        class="material-icons right">arrow_drop_down</i></a></li>
                <li><a href="/2016/dream">@DREAM</a></li>

                <li><a href="/2016/committee.html">Committees</a></li>
                <li><a href="/2016/previousYears/">Previous Years</a></li>
            </ul>
            <ul class="side-nav" id="mobile">
                <li><a href="/2016/ieeevis/">@IEEE VIS 2016</a>
                    <ul>
                        <li><a href="/2016/cfp_vis">Call for Participation</a></li>
                        <li><a href="/2016/papers">Paper Submission</a></li>
                        <li><a href="/2016/posters_ieee">Poster Submission</a></li>
                        <li><a href="/2016/designContest_vis">Design Contest</a></li>
                        <li><a href="/2016/program_ieee">Program</a></li>
                    </ul>
                </li>

                <li><a href="/2016/ismb/">@ISMB 2016</a>
                    <ul>
                        <li><a href="/2016/cfp_ismb">Call for Participation</a></li>
                        <li><a href="/2016/submission_ismb">Abstract Submission</a></li>
                        <li><a href="/2016/program_ismb">Program</a></li>
                    </ul>
                </li>
                <li><a href="/2016/previousYears/">Previous Years</a></li>
            </ul>
        </div>
    </nav>


    <div class="breadcrumbs moduletable">
  <ul class="breadcrumb">
    <li class="active">You are here: &nbsp;</li>
    <li><a href="/2016/" class="pathway">Home</a> |</li>
    
    <li><a href="/2016/ieeevis" class="pathway">IEEE VIS 2016</a> |</li>
    
    <li class="active"><span>Papers BioVis@VIS</span></li>
  </ul>
</div>

<div class="blue-section">
  <h3 itemprop="name">Papers BioVis@VIS</h3>
</div>

<div class="container">
<!--**Attention paper Authors:**
When preparing accepted papers please note that your paper should not exceed the following dimensions: *46 inches wide by 45 inches high*. There will be 2 papers per side on the each paper board. One paper will be an odd number and the other will be an even number. View a diagram of the the paper board in pdf format [here](http://www.iscb.org/images/stories/ismb2016/downloads/ISMB2016-paperSampler.pdf).
-->

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0109-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0109-file1.png" alt="Unboxing Cluster Heatmaps" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: Unboxing Cluster Heatmaps</div>
    <div><span class="tspeaker">Sophie Engle*, Sean Whalen, Alark Joshi, Katherine Pollard</span></div>
    <div>
      <p>Background: Cluster heatmaps are commonly used in biology and related fields to reveal hierarchical clusters in data matrices. This visualization technique has high data density and reveal clusters better than unordered heatmaps alone. However, cluster heatmaps have known issues making them both time consuming to use and prone to error. We hypothesize that visualization techniques without the rigid grid constraint of cluster heatmaps will perform better at clustering-related tasks. Results: We developed an approach to "unbox" the heatmap values and embed them directly in the hierarchical clustering results, allowing us to use standard hierarchical visualization techniques as alternatives to cluster heatmaps. We then tested our hypothesis by conducting a survey of 45 practitioners to determine how cluster heatmaps are used, prototyping alternatives to cluster heatmaps using pair analytics with a computational biologist, and evaluating those alternatives with hour-long interviews of 5 practitioners and an Amazon Mechanical Turk user study with approximately 200 participants. We found statistically significant performance differences for most clustering-related tasks, and in the number of perceived visual clusters. Visit git.io/vw0t3 for our results. Conclusions: The optimal technique varied by task. However, gapmaps were preferred by the interviewed practitioners and outperformed or performed as well as cluster heatmaps for clustering-related tasks. Gapmaps are similar to cluster heatmaps, but relax the heatmap grid constraints by introducing gaps between rows and/or columns that are not closely clustered. Based on these results, we recommend users adopt gapmaps as an alternative to cluster heatmaps.</p>
    </div>
    <!--<div><span><a href="/2016/files/Background: Cluster heatmaps are commonly used in biology and related fields to reveal hierarchical clusters in data matrices. This visualization technique has high data density and reveal clusters better than unordered heatmaps alone. However, cluster heatmaps have known issues making them both time consuming to use and prone to error. We hypothesize that visualization techniques without the rigid grid constraint of cluster heatmaps will perform better at clustering-related tasks. Results: We developed an approach to "unbox" the heatmap values and embed them directly in the hierarchical clustering results, allowing us to use standard hierarchical visualization techniques as alternatives to cluster heatmaps. We then tested our hypothesis by conducting a survey of 45 practitioners to determine how cluster heatmaps are used, prototyping alternatives to cluster heatmaps using pair analytics with a computational biologist, and evaluating those alternatives with hour-long interviews of 5 practitioners and an Amazon Mechanical Turk user study with approximately 200 participants. We found statistically significant performance differences for most clustering-related tasks, and in the number of perceived visual clusters. Visit git.io/vw0t3 for our results. Conclusions: The optimal technique varied by task. However, gapmaps were preferred by the interviewed practitioners and outperformed or performed as well as cluster heatmaps for clustering-related tasks. Gapmaps are similar to cluster heatmaps, but relax the heatmap grid constraints by introducing gaps between rows and/or columns that are not closely clustered. Based on these results, we recommend users adopt gapmaps as an alternative to cluster heatmaps.">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0121-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0121-file1.png" alt="PRODIGEN: Visualizing the Probability Landscape of Stochastic Gene Regulatory Networks in State and Time Space" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: PRODIGEN: Visualizing the Probability Landscape of Stochastic Gene Regulatory Networks in State and Time Space</div>
    <div><span class="tspeaker">Chihua Ma*, Timothy Luciani, Anna Terebus, Jie Liang, G.Elisabeta Marai</span></div>
    <div>
      <p>Background: Visualizing the complex probability landscape of stochastic gene regulatory networks can further biologists understanding of phenotypic behavior associated with specific genes. Results: We present PRODIGEN (PRObability DIstribution of GEne Networks), a web-based visual analysis tool for the systematic exploration of probability distributions over simulation time and state space in such networks. PRODIGEN was designed in collaboration with bioinformaticians who research stochastic gene networks. The analysis tool combines in a novel way existing, expanded, and new visual encodings to capture the time-varying characteristics of probability distributions: spaghetti plots over one dimensional projection, heatmaps of distributions over 2D projections, enhanced with overlaid time curves to display temporal changes, and novel individual glyphs of state information corresponding to particular peaks. Conclusions: We demonstrate the effectiveness of the tool through two case studies on the computed probabilistic landscape of a gene regulatory network and of a toggle-switch network. Domain expert feedback indicates that our visual approach can help biologists: 1) visualize probabilities of stable states, 2) explore the temporal probability distributions, and 3) discover small peaks in the probability landscape that have potential relation to specific diseases.</p>
    </div>
    <!--<div><span><a href="/2016/files/Background: Visualizing the complex probability landscape of stochastic gene regulatory networks can further biologists understanding of phenotypic behavior associated with specific genes. Results: We present PRODIGEN (PRObability DIstribution of GEne Networks), a web-based visual analysis tool for the systematic exploration of probability distributions over simulation time and state space in such networks. PRODIGEN was designed in collaboration with bioinformaticians who research stochastic gene networks. The analysis tool combines in a novel way existing, expanded, and new visual encodings to capture the time-varying characteristics of probability distributions: spaghetti plots over one dimensional projection, heatmaps of distributions over 2D projections, enhanced with overlaid time curves to display temporal changes, and novel individual glyphs of state information corresponding to particular peaks. Conclusions: We demonstrate the effectiveness of the tool through two case studies on the computed probabilistic landscape of a gene regulatory network and of a toggle-switch network. Domain expert feedback indicates that our visual approach can help biologists: 1) visualize probabilities of stable states, 2) explore the temporal probability distributions, and 3) discover small peaks in the probability landscape that have potential relation to specific diseases.">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0118-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0118-file1.png" alt="Generalized Box-Plot for Root Growth Ensembles" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: Generalized Box-Plot for Root Growth Ensembles</div>
    <div><span class="tspeaker">Viktor Vad*, Douglas Cedrim, Wolfgang Busch, Peter Filzmoser, Ivan Viola</span></div>
    <div>
      <p>Background: In the field of root biology there has been a remarkable progress in root phenotyping, which is the efficient acquisition and quantitative description of root morphology. What is currently missing are means to efficiently explore, exchange and present the massive amount of acquired, and often time dependent root phenotypes. Results: In this work, we present visual summaries of root ensembles by aggregating root images with identical genetic characteristics. We use the generalized box plot concept with a new formulation of data depth. In addition to spatial distributions, we created a visual representation to encode temporal distributions associated with the development of root individuals. Conclusions: The new formulation of data depth allows for much faster implementation close to interactive frame rates. This allows us to present the statistics from bootstrapping that characterize the root sample set quality. As a positive side effect of the new data-depth formulation we are able to define the geometric median for the curve ensemble, which was well received by the domain experts.</p>
    </div>
    <!--<div><span><a href="/2016/files/Background: In the field of root biology there has been a remarkable progress in root phenotyping, which is the efficient acquisition and quantitative description of root morphology. What is currently missing are means to efficiently explore, exchange and present the massive amount of acquired, and often time dependent root phenotypes. Results: In this work, we present visual summaries of root ensembles by aggregating root images with identical genetic characteristics. We use the generalized box plot concept with a new formulation of data depth. In addition to spatial distributions, we created a visual representation to encode temporal distributions associated with the development of root individuals. Conclusions: The new formulation of data depth allows for much faster implementation close to interactive frame rates. This allows us to present the statistics from bootstrapping that characterize the root sample set quality. As a positive side effect of the new data-depth formulation we are able to define the geometric median for the curve ensemble, which was well received by the domain experts.">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0111-file1.jpg"> <img style="padding-right: 10px;" src="/2016/files/paper0111-file1.jpg" alt="A Taxonomy of Visualization Tasks for the Analysis of Biological Pathway Data" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: A Taxonomy of Visualization Tasks for the Analysis of Biological Pathway Data</div>
    <div><span class="tspeaker">Paul Murray*, Fintan McGee, Angus Forbes</span></div>
    <div>
      <p>Background: Understanding complicated networks of interactions and chemical components is essential to solving contemporary problems in modern biology, especially in domains such as cancer and systems research. In these domains, biological pathway data is used to represent chains of interactions that occur within a given biological process. Visual representations can help researchers understand, interact with, and reason about these complex pathways in a number of ways. At the same time, these datasets offer unique challenges for visualization, due to their complexity and heterogeneity. Results: Here, we present taxonomy of tasks that are regularly performed by researchers who work with biological pathway data. The generation of these tasks was done in conjunction with interviews with several domain experts in biology. These tasks require further classification than is provided by existing taxonomies. We also examine existing visualization techniques that support each task, and we discuss gaps in the existing visualization space revealed by our taxonomy. Conclusions: Our taxonomy is designed to support the development and design of future biological pathway visualization applications. We conclude by suggesting future research directions based on our taxonomy and motivated by the comments received by our domain experts.</p>
    </div>
    <!--<div><span><a href="/2016/files/Background: Understanding complicated networks of interactions and chemical components is essential to solving contemporary problems in modern biology, especially in domains such as cancer and systems research. In these domains, biological pathway data is used to represent chains of interactions that occur within a given biological process. Visual representations can help researchers understand, interact with, and reason about these complex pathways in a number of ways. At the same time, these datasets offer unique challenges for visualization, due to their complexity and heterogeneity. Results: Here, we present taxonomy of tasks that are regularly performed by researchers who work with biological pathway data. The generation of these tasks was done in conjunction with interviews with several domain experts in biology. These tasks require further classification than is provided by existing taxonomies. We also examine existing visualization techniques that support each task, and we discuss gaps in the existing visualization space revealed by our taxonomy. Conclusions: Our taxonomy is designed to support the development and design of future biological pathway visualization applications. We conclude by suggesting future research directions based on our taxonomy and motivated by the comments received by our domain experts.">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0129-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0129-file1.png" alt="Comparative Visualization of Protein Secondary Structures" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: Comparative Visualization of Protein Secondary Structures</div>
    <div><span class="tspeaker">Lucia Kocincova, Miroslava Jaresova*, Jan Byska, Julius Parulek, Helwig Hauser, Barbora Kozlikov</span></div>
    <div>
      <p>Background: Protein function is determined by many factors, namely by its constitution, spatial arrangement, and dynamic behavior. Studying these factors helps the biochemists and biologists to better understand the protein behavior and to design proteins with modified properties. One of the most common approaches to these studies is to compare the protein structure with other molecules and to reveal similarities and differences in their polypeptide chains. Results: We support the comparison process by proposing a new visualization technique that bridges the gap between traditionally used 1D and 3D representations. By introducing the information about mutual positions of protein chains into the 1D sequential representation the users are able to observe the spatial differences between the proteins without any occlusion commonly present in 3D view. Our representation is designed to serve namely for comparison of multiple proteins or a set of time steps of molecular dynamics simulation. Conclusions: The novel representation is demonstrated on two case studies. The first study aims to compare a set of proteins from the family of cytochromes P450 where the position of the secondary structures has a significant impact on the substrate channeling. The second study focuses on the protein flexibility when by comparing a set of time steps our representation helps to reveal the most dynamically changing parts of the protein chain.</p>
    </div>
    <!--<div><span><a href="/2016/files/Background: Protein function is determined by many factors, namely by its constitution, spatial arrangement, and dynamic behavior. Studying these factors helps the biochemists and biologists to better understand the protein behavior and to design proteins with modified properties. One of the most common approaches to these studies is to compare the protein structure with other molecules and to reveal similarities and differences in their polypeptide chains. Results: We support the comparison process by proposing a new visualization technique that bridges the gap between traditionally used 1D and 3D representations. By introducing the information about mutual positions of protein chains into the 1D sequential representation the users are able to observe the spatial differences between the proteins without any occlusion commonly present in 3D view. Our representation is designed to serve namely for comparison of multiple proteins or a set of time steps of molecular dynamics simulation. Conclusions: The novel representation is demonstrated on two case studies. The first study aims to compare a set of proteins from the family of cytochromes P450 where the position of the secondary structures has a significant impact on the substrate channeling. The second study focuses on the protein flexibility when by comparing a set of time steps our representation helps to reveal the most dynamically changing parts of the protein chain.">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0115-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0115-file1.png" alt="Bio-physically Plausible Visualization of Highly Scattering Fluorescent Neocortical Models for in Silico Experimentation" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: Bio-physically Plausible Visualization of Highly Scattering Fluorescent Neocortical Models for in Silico Experimentation</div>
    <div><span class="tspeaker">Marwan Abdellah*, Ahmet Bilgili, Stefan Eilemann, Julian Shillcock, Henry Markram, Felix Schurmann</span></div>
    <div>
      <p>Background: We present a visualization pipeline capable of accurate rendering of highly scattering fluorescent neocortical neuronal models. The pipeline is mainly developed to serve the computational neurobiology community. It allows the scientists to visualize the results of their virtual experiments that are performed in computer simulations, or in silico. The impact of the presented pipeline opens novel avenues for assisting the neuroscientists to build biologically accurate models of the brain. These models result from computer simulations of physical experiments that use fluorescence imaging to understand the structural and functional aspects of the brain. Due to the limited capabilities of the current visualization workflows to handle fluorescent volumetric datasets, we propose a physically-based optical model that can accurately simulate light interaction with fluorescent-tagged scattering media based on the basic principles of geometric optics and Monte Carlo path tracing. We also develop an automated and efficient framework for generating dense fluorescent tissue blocks from a neocortical column model that is composed of approximately 31000 neurons.  Results: Our pipeline is used to visualize a virtual fluorescent tissue block of 50 m 3 that is reconstructed from the somatosensory cortex of juvenile rat. The fluorescence optical model is qualitatively analyzed and validated against experimental emission spectra of different fluorescent dyes from the Alexa Fluor family. Keywords: Modeling and Simulation; Highly Scattering Volumes; Fluorescence Rendering and Visualization; Neocortical Brain Models; In Silico Neuroscience AMS Subject Classification: Modelling and Simulation</p>
    </div>
    <!--<div><span><a href="/2016/files/Background: We present a visualization pipeline capable of accurate rendering of highly scattering fluorescent neocortical neuronal models. The pipeline is mainly developed to serve the computational neurobiology community. It allows the scientists to visualize the results of their virtual experiments that are performed in computer simulations, or in silico. The impact of the presented pipeline opens novel avenues for assisting the neuroscientists to build biologically accurate models of the brain. These models result from computer simulations of physical experiments that use fluorescence imaging to understand the structural and functional aspects of the brain. Due to the limited capabilities of the current visualization workflows to handle fluorescent volumetric datasets, we propose a physically-based optical model that can accurately simulate light interaction with fluorescent-tagged scattering media based on the basic principles of geometric optics and Monte Carlo path tracing. We also develop an automated and efficient framework for generating dense fluorescent tissue blocks from a neocortical column model that is composed of approximately 31000 neurons.  Results: Our pipeline is used to visualize a virtual fluorescent tissue block of 50 m 3 that is reconstructed from the somatosensory cortex of juvenile rat. The fluorescence optical model is qualitatively analyzed and validated against experimental emission spectra of different fluorescent dyes from the Alexa Fluor family. Keywords: Modeling and Simulation; Highly Scattering Volumes; Fluorescence Rendering and Visualization; Neocortical Brain Models; In Silico Neuroscience AMS Subject Classification: Modelling and Simulation">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0119-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0119-file1.png" alt="Virtual Reality Assisted Microscopy Data Visualization and Colocalization Analysis" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: Virtual Reality Assisted Microscopy Data Visualization and Colocalization Analysis</div>
    <div><span class="tspeaker">Rensu Theart*, Ben Loos, Thomas Niesler</span></div>
    <div>
      <p>Background:  Confocal microscopes deliver detailed three-dimensional data and are instrumental in biological analysis and research. Usually, this three-dimensional data is rendered as a projection onto a two-dimensional display. We describe a system for rendering such data using a modern virtual reality (VR) headset. Sample manipulation is possible by fully-immersive hand-tracking and also by means of a conventional gamepad. We apply this system to the specific task of colocalization analysis, an important analysis tool in biological microscopy. We evaluate our system by means of a set of user trials. Results:  The user trials show that, despite inaccuracies which still plague the hand tracking, this is the most productive and intuitive interface. The inaccuracies nevertheless lead to a perception among users that productivity is low, resulting in a subjective preference for the gamepad. Fully-immersive manipulation was shown to be particularly effective when defining a region of interest (ROI) for colocalization analysis. Conclusions:  Virtual reality offers an attractive and powerful means of visualization for microscopy data. Fully immersive interfaces using hand tracking show the highest levels of intuitiveness and consequent productivity. However, current inaccuracies in hand tracking performance still lead to a disproportionately critical user perception.</p>
    </div>
    <!--<div><span><a href="/2016/files/Background:  Confocal microscopes deliver detailed three-dimensional data and are instrumental in biological analysis and research. Usually, this three-dimensional data is rendered as a projection onto a two-dimensional display. We describe a system for rendering such data using a modern virtual reality (VR) headset. Sample manipulation is possible by fully-immersive hand-tracking and also by means of a conventional gamepad. We apply this system to the specific task of colocalization analysis, an important analysis tool in biological microscopy. We evaluate our system by means of a set of user trials. Results:  The user trials show that, despite inaccuracies which still plague the hand tracking, this is the most productive and intuitive interface. The inaccuracies nevertheless lead to a perception among users that productivity is low, resulting in a subjective preference for the gamepad. Fully-immersive manipulation was shown to be particularly effective when defining a region of interest (ROI) for colocalization analysis. Conclusions:  Virtual reality offers an attractive and powerful means of visualization for microscopy data. Fully immersive interfaces using hand tracking show the highest levels of intuitiveness and consequent productivity. However, current inaccuracies in hand tracking performance still lead to a disproportionately critical user perception.">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>

<div class="talk">
  <table>
  <tr>
    <td width="300px">
      <a href="/2016/files/paper0128-file1.png"> <img style="padding-right: 10px;" src="/2016/files/paper0128-file1.png" alt="Interactive Exploration of Ligand Transportation through Protein Tunnels" height="250" width="250" /></a>
    </td>
  <td>
    <div class="ttitle">paper: Interactive Exploration of Ligand Transportation through Protein Tunnels</div>
    <div><span class="tspeaker">Katarina Furmanova*, Miroslava Jaresova, Jan Byska, Adam Jurcik, Julius Parulek, Helwig Hauser, Barbora Kozlikova</span></div>
    <div>
      <p>Background: Protein structures and their interaction with ligands have been in the focus of biochemistry and structural biology research for decades. The transportation of ligand into the protein active site is often complex process, driven by geometric and physico-chemical properties, which renders the ligand path full of jitter and impasses. This prevents understanding of the ligand transportation and reasoning behind its behavior along the path. Results: To address the needs of the domain experts we design an explorative visualization solution based on a multi-scale simplification model. It helps to navigate the user to the most interesting parts of the ligand trajectory by exploring different attributes of the ligand and its movement, such as its distance to the active site, changes of amino acids lining the ligand, or ligand "stuckness". The process is supported by three linked views -- 3D representation of the simplified trajectory, scatterplot matrix, and bar charts with line representation of ligand-lining amino acids. Conclusions: The usage of our tool is demonstrated on molecular dynamics simulations provided by the domain experts. The tool was tested by the domain experts from protein engineering and the results confirm that it helps to navigate the user to the most interesting parts of the ligand trajectory and to understand the ligand behavior. </p>
    </div>
    <!--<div><span><a href="/2016/files/Background: Protein structures and their interaction with ligands have been in the focus of biochemistry and structural biology research for decades. The transportation of ligand into the protein active site is often complex process, driven by geometric and physico-chemical properties, which renders the ligand path full of jitter and impasses. This prevents understanding of the ligand transportation and reasoning behind its behavior along the path. Results: To address the needs of the domain experts we design an explorative visualization solution based on a multi-scale simplification model. It helps to navigate the user to the most interesting parts of the ligand trajectory by exploring different attributes of the ligand and its movement, such as its distance to the active site, changes of amino acids lining the ligand, or ligand "stuckness". The process is supported by three linked views -- 3D representation of the simplified trajectory, scatterplot matrix, and bar charts with line representation of ligand-lining amino acids. Conclusions: The usage of our tool is demonstrated on molecular dynamics simulations provided by the domain experts. The tool was tested by the domain experts from protein engineering and the results confirm that it helps to navigate the user to the most interesting parts of the ligand trajectory and to understand the ligand behavior. ">Download Full Abstract</a></span></div>-->
  </td>
  </tr>
  </table>
</div>


</div>



<footer class="page-footer">
    <div class="container">
        <div class="row">
            <div class="col l4 m4 s12">
                <img src="/2016/images/logo-white.svg" width="140">
            </div>

            <div class="col l4 m4 s12">
                <h5 class="white-text">Sponsors</h5>
                <ul>
                    <li><a href="https://www.rstudio.com/" target="_blank">RStudio</a></li>
                    <li><a href="https://www.sbgenomics.com/" target="_blank">Seven Bridges</a></li>
                    <li><a href="http://www.nvidia.com/content/global/global.php" target="_blank">nvidia</a></li>
                    <li><a href="http://f1000research.com/" target="_blank">F1000</a></li>
                </ul>
            </div>

            <div class="col l4 m4 s12">
                <h5 class="white-text">Links</h5>
                <ul>
                    <li><a href="http://ieeevis.org/" target="_blank">IEEE VIS</a></li>
                    <li><a href="https://www.iscb.org/ismb2016" target="_blank">ISMB</a></li>
                    <li><a href="http://www.biomedcentral.com/bmcbioinformatics/series/BioVis2014" target="_blank">BMC
                        Thematic Series</a></li>
                </ul>
            </div>
        </div>

        <div class="footer-copyright center-align">

            © 6th Symposium on Biological Data Visualization | Designed by <a href="http://www.antarctic-design.co.uk"
                                                                              target="_blank">Antarctic Design</a>.


        </div>
    </div>

</footer>

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/js/materialize.min.js"></script>


<script>
    $(".button-collapse").sideNav();
    $(".dropdown-button").dropdown();
</script>




<!-- Google Analytics -->
<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-22015250-1', 'auto');
    ga('send', 'pageview');



    (function (){
        var links = document.getElementsByTagName('a');
        var myLink = null;
        var startWith = null;

        for(var i = 0; i < links.length; ++i) {
            myLink = links[i];

            startWith = myLink.getAttribute('href').slice(0, 1);

            if (startWith!="/" && startWith!="#"){
                addListener(myLink,'mousedown',function(){

                    console.log("a:", this.getAttribute('href'))
                    ga('send', 'event', 'link', 'click', this.getAttribute('href'));

                })
            }

        }
    })();



    /**
     * Utility to wrap the different behaviors between W3C-compliant browsers
     * and IE when adding event handlers.
     *
     * @param {Object} element Object on which to attach the event listener.
     * @param {string} type A string representing the event type to listen for
     *     (e.g. load, click, etc.).
     * @param {function()} callback The function that receives the notification.
     */
    function addListener(element, type, callback) {
        if (element.addEventListener) element.addEventListener(type, callback);
        else if (element.attachEvent) element.attachEvent('on' + type, callback);
    }




</script>
<!-- End Google Analytics -->


</body>
</html>
